<html>

<head>
<meta http-equiv=Content-Type content="text/html; charset=windows-1252">
<meta name=Generator content="Microsoft Word 15 (filtered)">
<title>nmrML spec document</title>
<style>
<!--
 /* Font Definitions */
 @font-face
	{font-family:Helvetica;
	panose-1:2 11 6 4 2 2 2 2 2 4;}
@font-face
	{font-family:Wingdings;
	panose-1:5 0 0 0 0 0 0 0 0 0;}
@font-face
	{font-family:"MS Mincho";
	panose-1:2 2 6 9 4 2 5 8 3 4;}
@font-face
	{font-family:"MS Mincho";
	panose-1:2 2 6 9 4 2 5 8 3 4;}
@font-face
	{font-family:Calibri;
	panose-1:2 15 5 2 2 2 4 3 2 4;}
@font-face
	{font-family:"Droid Sans Fallback";
	panose-1:0 0 0 0 0 0 0 0 0 0;}
@font-face
	{font-family:"Lohit Hindi";
	panose-1:0 0 0 0 0 0 0 0 0 0;}
@font-face
	{font-family:"Comic Sans MS";
	panose-1:3 15 7 2 3 3 2 2 2 4;}
@font-face
	{font-family:Tahoma;
	panose-1:2 11 6 4 3 5 4 4 2 4;}
@font-face
	{font-family:"\@MS Mincho";
	panose-1:2 2 6 9 4 2 5 8 3 4;}
 /* Style Definitions */
 p.MsoNormal, li.MsoNormal, div.MsoNormal
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:0cm;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
h1
	{margin-top:6.0pt;
	margin-right:0cm;
	margin-bottom:3.0pt;
	margin-left:18.0pt;
	text-indent:-18.0pt;
	line-height:107%;
	page-break-after:avoid;
	font-size:16.0pt;
	font-family:"Arial","sans-serif";
	font-weight:bold;}
h2
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:18.0pt;
	text-indent:-18.0pt;
	line-height:107%;
	page-break-after:avoid;
	font-size:14.0pt;
	font-family:"Arial","sans-serif";
	font-weight:normal;}
h3
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:18.0pt;
	text-indent:-18.0pt;
	line-height:107%;
	page-break-after:avoid;
	font-size:13.0pt;
	font-family:"Helvetica","sans-serif";
	font-weight:normal;}
h4
	{margin-top:12.0pt;
	margin-right:0cm;
	margin-bottom:3.0pt;
	margin-left:18.0pt;
	text-indent:-18.0pt;
	line-height:107%;
	page-break-after:avoid;
	font-size:14.0pt;
	font-family:"Times New Roman","serif";
	font-weight:bold;}
h5
	{margin-top:12.0pt;
	margin-right:0cm;
	margin-bottom:3.0pt;
	margin-left:18.0pt;
	text-indent:-18.0pt;
	line-height:107%;
	font-size:13.0pt;
	font-family:"Arial","sans-serif";
	font-weight:bold;
	font-style:italic;}
h6
	{margin-top:12.0pt;
	margin-right:0cm;
	margin-bottom:3.0pt;
	margin-left:18.0pt;
	text-indent:-18.0pt;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Times New Roman","serif";
	font-weight:bold;}
p.MsoHeading7, li.MsoHeading7, div.MsoHeading7
	{margin-top:12.0pt;
	margin-right:0cm;
	margin-bottom:3.0pt;
	margin-left:18.0pt;
	text-indent:-18.0pt;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Times New Roman","serif";}
p.MsoHeading8, li.MsoHeading8, div.MsoHeading8
	{margin-top:12.0pt;
	margin-right:0cm;
	margin-bottom:3.0pt;
	margin-left:18.0pt;
	text-indent:-18.0pt;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Times New Roman","serif";
	font-style:italic;}
p.MsoHeading9, li.MsoHeading9, div.MsoHeading9
	{margin-top:12.0pt;
	margin-right:0cm;
	margin-bottom:3.0pt;
	margin-left:18.0pt;
	text-indent:-18.0pt;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Arial","sans-serif";}
p.MsoIndex1, li.MsoIndex1, div.MsoIndex1
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:10.0pt;
	text-indent:-10.0pt;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.MsoIndex2, li.MsoIndex2, div.MsoIndex2
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:20.0pt;
	text-indent:-10.0pt;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.MsoIndex3, li.MsoIndex3, div.MsoIndex3
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:30.0pt;
	text-indent:-10.0pt;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.MsoIndex4, li.MsoIndex4, div.MsoIndex4
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:40.0pt;
	text-indent:-10.0pt;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.MsoIndex5, li.MsoIndex5, div.MsoIndex5
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:50.0pt;
	text-indent:-10.0pt;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.MsoIndex6, li.MsoIndex6, div.MsoIndex6
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:60.0pt;
	text-indent:-10.0pt;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.MsoIndex7, li.MsoIndex7, div.MsoIndex7
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:70.0pt;
	text-indent:-10.0pt;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.MsoIndex8, li.MsoIndex8, div.MsoIndex8
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:80.0pt;
	text-indent:-10.0pt;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.MsoIndex9, li.MsoIndex9, div.MsoIndex9
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:90.0pt;
	text-indent:-10.0pt;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.MsoToc1, li.MsoToc1, div.MsoToc1
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:5.0pt;
	margin-left:0cm;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.MsoToc2, li.MsoToc2, div.MsoToc2
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:5.0pt;
	margin-left:12.0pt;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.MsoToc3, li.MsoToc3, div.MsoToc3
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:5.0pt;
	margin-left:24.0pt;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.MsoNormalIndent, li.MsoNormalIndent, div.MsoNormalIndent
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:36.0pt;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.MsoFootnoteText, li.MsoFootnoteText, div.MsoFootnoteText
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:0cm;
	line-height:107%;
	font-size:10.0pt;
	font-family:"Arial","sans-serif";}
p.MsoCommentText, li.MsoCommentText, div.MsoCommentText
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:0cm;
	line-height:107%;
	font-size:10.0pt;
	font-family:"Arial","sans-serif";}
p.MsoHeader, li.MsoHeader, div.MsoHeader
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:0cm;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.MsoFooter, li.MsoFooter, div.MsoFooter
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:0cm;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.MsoIndexHeading, li.MsoIndexHeading, div.MsoIndexHeading
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:0cm;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";
	font-weight:bold;}
p.MsoCaption, li.MsoCaption, div.MsoCaption
	{margin-top:6.0pt;
	margin-right:0cm;
	margin-bottom:6.0pt;
	margin-left:0cm;
	line-height:107%;
	font-size:10.0pt;
	font-family:"Arial","sans-serif";
	font-weight:bold;}
p.MsoTof, li.MsoTof, div.MsoTof
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:20.0pt;
	text-indent:-20.0pt;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.MsoEnvelopeAddress, li.MsoEnvelopeAddress, div.MsoEnvelopeAddress
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:144.0pt;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.MsoEnvelopeReturn, li.MsoEnvelopeReturn, div.MsoEnvelopeReturn
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:0cm;
	line-height:107%;
	font-size:10.0pt;
	font-family:"Arial","sans-serif";}
span.MsoFootnoteReference
	{vertical-align:super;}
span.MsoEndnoteReference
	{vertical-align:super;}
p.MsoEndnoteText, li.MsoEndnoteText, div.MsoEndnoteText
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:0cm;
	line-height:107%;
	font-size:10.0pt;
	font-family:"Arial","sans-serif";}
p.MsoToa, li.MsoToa, div.MsoToa
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:10.0pt;
	text-indent:-10.0pt;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.MsoMacroText, li.MsoMacroText, div.MsoMacroText
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:0cm;
	line-height:107%;
	font-size:10.0pt;
	font-family:"Courier New";}
p.MsoToaHeading, li.MsoToaHeading, div.MsoToaHeading
	{margin-top:6.0pt;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:0cm;
	margin-bottom:.0001pt;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";
	font-weight:bold;}
p.MsoList, li.MsoList, div.MsoList
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:18.0pt;
	text-indent:-18.0pt;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.MsoListBullet, li.MsoListBullet, div.MsoListBullet
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:18.0pt;
	text-indent:-18.0pt;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.MsoListNumber, li.MsoListNumber, div.MsoListNumber
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:18.0pt;
	text-indent:-18.0pt;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.MsoList2, li.MsoList2, div.MsoList2
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:36.0pt;
	text-indent:-18.0pt;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.MsoList3, li.MsoList3, div.MsoList3
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:54.0pt;
	text-indent:-18.0pt;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.MsoList4, li.MsoList4, div.MsoList4
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:72.0pt;
	text-indent:-18.0pt;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.MsoList5, li.MsoList5, div.MsoList5
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:90.0pt;
	text-indent:-18.0pt;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.MsoListBullet2, li.MsoListBullet2, div.MsoListBullet2
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:36.0pt;
	text-indent:-18.0pt;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.MsoListBullet3, li.MsoListBullet3, div.MsoListBullet3
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:54.0pt;
	text-indent:-18.0pt;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.MsoListBullet4, li.MsoListBullet4, div.MsoListBullet4
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:72.0pt;
	text-indent:-18.0pt;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.MsoListBullet5, li.MsoListBullet5, div.MsoListBullet5
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:90.0pt;
	text-indent:-18.0pt;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.MsoListNumber2, li.MsoListNumber2, div.MsoListNumber2
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:36.0pt;
	text-indent:-18.0pt;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.MsoListNumber3, li.MsoListNumber3, div.MsoListNumber3
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:54.0pt;
	text-indent:-18.0pt;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.MsoListNumber4, li.MsoListNumber4, div.MsoListNumber4
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:72.0pt;
	text-indent:-18.0pt;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.MsoListNumber5, li.MsoListNumber5, div.MsoListNumber5
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:90.0pt;
	text-indent:-18.0pt;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.MsoTitle, li.MsoTitle, div.MsoTitle
	{margin-top:12.0pt;
	margin-right:0cm;
	margin-bottom:3.0pt;
	margin-left:0cm;
	text-align:center;
	line-height:107%;
	font-size:16.0pt;
	font-family:"Arial","sans-serif";
	font-weight:bold;}
p.MsoClosing, li.MsoClosing, div.MsoClosing
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:216.0pt;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.MsoSignature, li.MsoSignature, div.MsoSignature
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:216.0pt;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.MsoBodyTextIndent, li.MsoBodyTextIndent, div.MsoBodyTextIndent
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:6.0pt;
	margin-left:0cm;
	text-indent:10.5pt;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.MsoListContinue, li.MsoListContinue, div.MsoListContinue
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:6.0pt;
	margin-left:18.0pt;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.MsoListContinue2, li.MsoListContinue2, div.MsoListContinue2
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:6.0pt;
	margin-left:36.0pt;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.MsoListContinue3, li.MsoListContinue3, div.MsoListContinue3
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:6.0pt;
	margin-left:54.0pt;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.MsoListContinue4, li.MsoListContinue4, div.MsoListContinue4
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:6.0pt;
	margin-left:72.0pt;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.MsoListContinue5, li.MsoListContinue5, div.MsoListContinue5
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:6.0pt;
	margin-left:90.0pt;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.MsoMessageHeader, li.MsoMessageHeader, div.MsoMessageHeader
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:54.0pt;
	text-indent:-54.0pt;
	line-height:107%;
	background:#CCCCCC;
	border:none;
	padding:0cm;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.MsoSubtitle, li.MsoSubtitle, div.MsoSubtitle
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:3.0pt;
	margin-left:0cm;
	text-align:center;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.MsoDate, li.MsoDate, div.MsoDate
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:0cm;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.MsoBodyTextFirstIndent2, li.MsoBodyTextFirstIndent2, div.MsoBodyTextFirstIndent2
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:6.0pt;
	margin-left:18.0pt;
	text-indent:10.5pt;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.MsoNoteHeading, li.MsoNoteHeading, div.MsoNoteHeading
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:0cm;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.MsoBodyText2, li.MsoBodyText2, div.MsoBodyText2
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:6.0pt;
	margin-left:0cm;
	line-height:200%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.MsoBodyText3, li.MsoBodyText3, div.MsoBodyText3
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:6.0pt;
	margin-left:0cm;
	line-height:107%;
	font-size:8.0pt;
	font-family:"Arial","sans-serif";}
p.MsoBodyTextIndent2, li.MsoBodyTextIndent2, div.MsoBodyTextIndent2
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:6.0pt;
	margin-left:18.0pt;
	line-height:200%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.MsoBodyTextIndent3, li.MsoBodyTextIndent3, div.MsoBodyTextIndent3
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:6.0pt;
	margin-left:18.0pt;
	line-height:107%;
	font-size:8.0pt;
	font-family:"Arial","sans-serif";}
p.MsoBlockText, li.MsoBlockText, div.MsoBlockText
	{margin-top:0cm;
	margin-right:36.0pt;
	margin-bottom:8.0pt;
	margin-left:18.0pt;
	line-height:107%;
	font-size:7.0pt;
	font-family:"Courier New";}
a:link, span.MsoHyperlink
	{color:#0563C1;
	text-decoration:underline;}
a:visited, span.MsoHyperlinkFollowed
	{color:purple;
	text-decoration:underline;}
p.MsoDocumentMap, li.MsoDocumentMap, div.MsoDocumentMap
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:0cm;
	line-height:107%;
	background:navy;
	font-size:12.0pt;
	font-family:"Tahoma","sans-serif";}
p.MsoPlainText, li.MsoPlainText, div.MsoPlainText
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:36.0pt;
	line-height:107%;
	font-size:10.0pt;
	font-family:"Courier New";}
p.MsoAutoSig, li.MsoAutoSig, div.MsoAutoSig
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:0cm;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:0cm;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Times New Roman","serif";}
address
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:0cm;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";
	font-style:italic;}
pre
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:0cm;
	line-height:107%;
	font-size:7.0pt;
	font-family:"Courier New";}
p.MsoCommentSubject, li.MsoCommentSubject, div.MsoCommentSubject
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:0cm;
	line-height:107%;
	font-size:10.0pt;
	font-family:"Arial","sans-serif";
	font-weight:bold;}
p.MsoAcetate, li.MsoAcetate, div.MsoAcetate
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:0cm;
	line-height:107%;
	font-size:8.0pt;
	font-family:"Tahoma","sans-serif";}
p.MsoRMPane, li.MsoRMPane, div.MsoRMPane
	{margin:0cm;
	margin-bottom:.0001pt;
	line-height:normal;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.MsoListParagraph, li.MsoListParagraph, div.MsoListParagraph
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:36.0pt;
	margin-bottom:.0001pt;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.MsoListParagraphCxSpFirst, li.MsoListParagraphCxSpFirst, div.MsoListParagraphCxSpFirst
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:36.0pt;
	margin-bottom:.0001pt;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.MsoListParagraphCxSpMiddle, li.MsoListParagraphCxSpMiddle, div.MsoListParagraphCxSpMiddle
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:36.0pt;
	margin-bottom:.0001pt;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.MsoListParagraphCxSpLast, li.MsoListParagraphCxSpLast, div.MsoListParagraphCxSpLast
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:36.0pt;
	margin-bottom:.0001pt;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.MsoTocHeading, li.MsoTocHeading, div.MsoTocHeading
	{margin-top:12.0pt;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:0cm;
	margin-bottom:.0001pt;
	line-height:107%;
	page-break-after:avoid;
	font-size:16.0pt;
	font-family:"Calibri Light","sans-serif";
	color:#2E74B5;}
span.InternetLink
	{mso-style-name:"Internet Link";
	color:#0563C1;
	text-decoration:underline;}
span.bodytext
	{mso-style-name:bodytext;}
span.popup
	{mso-style-name:popup;}
span.EndnoteAnchor
	{mso-style-name:"Endnote Anchor";
	vertical-align:super;}
span.EndnotentextZchn
	{mso-style-name:"Endnotentext Zchn";
	font-family:"Arial","sans-serif";}
span.literal
	{mso-style-name:literal;}
span.ListLabel1
	{mso-style-name:"ListLabel 1";
	font-family:"Courier New";}
span.ListLabel2
	{mso-style-name:"ListLabel 2";
	font-family:Symbol;}
span.ListLabel3
	{mso-style-name:"ListLabel 3";
	font-family:Wingdings;}
span.ListLabel4
	{mso-style-name:"ListLabel 4";
	font-variant:normal !important;
	color:black;
	text-transform:none;
	position:relative;
	top:0pt;
	background:yellow;
	font-weight:normal;
	font-style:normal;
	text-decoration:none none;
	vertical-align:baseline;}
span.ListLabel5
	{mso-style-name:"ListLabel 5";}
span.ListLabel6
	{mso-style-name:"ListLabel 6";
	font-family:"Arial","sans-serif";
	font-variant:normal !important;
	color:black;
	text-transform:none;
	position:relative;
	top:0pt;
	font-weight:normal;
	font-style:normal;
	text-decoration:none none;
	vertical-align:baseline;}
span.EndnoteCharacters
	{mso-style-name:"Endnote Characters";}
span.IndexLink
	{mso-style-name:"Index Link";}
span.FootnoteAnchor
	{mso-style-name:"Footnote Anchor";
	vertical-align:super;}
span.FootnoteCharacters
	{mso-style-name:"Footnote Characters";}
p.Heading, li.Heading, div.Heading
	{mso-style-name:Heading;
	margin-top:12.0pt;
	margin-right:0cm;
	margin-bottom:6.0pt;
	margin-left:0cm;
	line-height:107%;
	page-break-after:avoid;
	font-size:14.0pt;
	font-family:"Arial","sans-serif";}
p.TextBody, li.TextBody, div.TextBody
	{mso-style-name:"Text Body";
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:6.0pt;
	margin-left:0cm;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.Index, li.Index, div.Index
	{mso-style-name:Index;
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:0cm;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.nobreak, li.nobreak, div.nobreak
	{mso-style-name:nobreak;
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:0cm;
	line-height:107%;
	page-break-after:avoid;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.HTMLBody, li.HTMLBody, div.HTMLBody
	{mso-style-name:"HTML Body";
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:0cm;
	line-height:107%;
	font-size:9.0pt;
	font-family:"Comic Sans MS";}
p.TextBodyIndent, li.TextBodyIndent, div.TextBodyIndent
	{mso-style-name:"Text Body Indent";
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:6.0pt;
	margin-left:18.0pt;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.ComplimentaryClose, li.ComplimentaryClose, div.ComplimentaryClose
	{mso-style-name:"Complimentary Close";
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:0cm;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.Contents1, li.Contents1, div.Contents1
	{mso-style-name:"Contents 1";
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:0cm;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.Contents2, li.Contents2, div.Contents2
	{mso-style-name:"Contents 2";
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:10.0pt;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.Contents3, li.Contents3, div.Contents3
	{mso-style-name:"Contents 3";
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:20.0pt;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.Contents4, li.Contents4, div.Contents4
	{mso-style-name:"Contents 4";
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:30.0pt;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.Contents5, li.Contents5, div.Contents5
	{mso-style-name:"Contents 5";
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:40.0pt;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.Contents6, li.Contents6, div.Contents6
	{mso-style-name:"Contents 6";
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:50.0pt;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.Contents7, li.Contents7, div.Contents7
	{mso-style-name:"Contents 7";
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:60.0pt;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.Contents8, li.Contents8, div.Contents8
	{mso-style-name:"Contents 8";
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:70.0pt;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.Contents9, li.Contents9, div.Contents9
	{mso-style-name:"Contents 9";
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:80.0pt;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
p.Appendix, li.Appendix, div.Appendix
	{mso-style-name:Appendix;
	margin-top:6.0pt;
	margin-right:0cm;
	margin-bottom:3.0pt;
	margin-left:36.0pt;
	text-indent:-18.0pt;
	line-height:107%;
	page-break-after:avoid;
	font-size:16.0pt;
	font-family:"Arial","sans-serif";
	font-weight:bold;}
p.Sub-appendix, li.Sub-appendix, div.Sub-appendix
	{mso-style-name:Sub-appendix;
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:17.85pt;
	text-indent:-17.85pt;
	line-height:107%;
	font-size:10.0pt;
	font-family:"Arial","sans-serif";}
p.StyleHTMLPreformatted8pt, li.StyleHTMLPreformatted8pt, div.StyleHTMLPreformatted8pt
	{mso-style-name:"Style HTML Preformatted + 8 pt";
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:0cm;
	line-height:107%;
	font-size:7.0pt;
	font-family:"Courier New";}
p.Endnote, li.Endnote, div.Endnote
	{mso-style-name:Endnote;
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:0cm;
	line-height:107%;
	font-size:12.0pt;
	font-family:"Arial","sans-serif";}
.MsoChpDefault
	{font-family:"Calibri","sans-serif";}
.MsoPapDefault
	{margin-bottom:8.0pt;
	line-height:107%;}
 /* Page Definitions */
 @page WordSection1
	{size:612.0pt 792.0pt;
	margin:72.0pt 54.0pt 36.0pt 90.0pt;}
div.WordSection1
	{page:WordSection1;}
 /* List Definitions */
 ol
	{margin-bottom:0cm;}
ul
	{margin-bottom:0cm;}
-->
</style>

</head>

<body lang=EN-US link="#0563C1" vlink=purple>

<div class=WordSection1>

<p class=MsoNormal align=center style='text-align:center'><b><img width=675
height=244 id="Grafik 12" src="NMR-ML1.0_specificationDoc-Dateien/image001.jpg"></b></p>

<p class=MsoNormal><b><span lang=EN-GB>&nbsp;</span></b></p>

<p class=MsoNormal align=center style='text-align:center'><b><span lang=EN-GB>NMR-ML:
Nuclear Magnetic Resonance Spectrometry Markup Language</span></b></p>

<p class=MsoNormal><b><span lang=EN-GB>&nbsp;</span></b></p>

<p class=MsoListParagraph align=center style='margin-left:21.0pt;text-align:
center;text-indent:-18.0pt'><span style='font-size:20.0pt;line-height:107%'>-<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp; </span></span><b><span
lang=EN-GB style='font-size:20.0pt;line-height:107%'>nmrML Specification
Document, Version 1.0 -</span></b></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal><u><span lang=EN-GB>Status of This Document</span></u></p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph'><span
lang=EN-GB>This document presents the first draft of a specification for the
nmrML XML data format developed by the COSMOS work package 2 ‘Standards
Development’ in approval of the Metabolomics Standards Initiative (MSI). Its
structure and wording was taken from an existing specification of equal scope,
namely the PSI mzML specification available under </span><a
href="http://www.psidev.info/mzml"><span class=InternetLink><span lang=EN-GB>http://www.psidev.info/mzml</span></span></a><span
lang=EN-GB>. Distribution is unlimited.</span></p>

<p class=MsoNormal>&nbsp;</p>

<h1 style='margin-left:0cm;text-indent:0cm'><a name="_Toc379880872"></a><a
name="_Toc375140733"></a><a name="_Toc156877855"></a><a name="_Toc118017561"></a><a
name="_Ref525097868"></a><span lang=EN-GB>Abstract</span></h1>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph'><span
lang=EN-GB>The COSMOS EU project and the Metabolomics Standards Initiative
(MSI) defines community standards for data representation in metabolomics to facilitate
data comparison, exchange and verification. The COSMOS WP2 develops standards
for describing NMR experiment data. This document presents information to the
NMR community about the modelling in XML of the experimental results obtained
by NMR of metabolite samples and biomolecular compounds. </span>COSMOS WP2 here
specifies the missing open standard called NMR Markup Language (nmrML) for
capturing and disseminating Nuclear Magnetic Resonance spectroscopy data in
metabolomics. This is urgently needed as long-term archival format if
metabolomic databases are to capture all the formats of metabolomics data, as
well as supporting developments in cheminformatics and structural biology. </p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph'><span
lang=EN-GB>This XML format is inspired by the PSI mzML format and hence
consists of an XML Schema Definition (XSD) that is accompanied by a controlled
vocabulary (CV). The XSD defines the allowed XML elements and attributes and
their corresponding XML structure. The CV provides the terminology to describe
the data in detail and provide standardized values for the XML tags.</span></p>

<p class=MsoNormal><span lang=EN-GB>This document assumes familiarity with a
few data modelling notations, namely XML and XML Schema (</span><a
href="http://www.w3.org/XML/Schema"><span class=InternetLink><span lang=EN-GB>www.w3.org/XML/Schema</span></span></a><span
lang=EN-GB>) for the XSD-based specification of the XML standard and controlled
vocabularies in which terms are described in an ‘is-a’ hierarchy called
subsumption graph or ‘taxonomy’. You should know what xml elements are, that
they can have attributes and that elements can be nested. You should know that
a CV term usually has an ID (accession), a label, and potentially more
metadata.</span></p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph'>&nbsp;</p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph'><span
lang=EN-GB>This document should be read in conjunction with a kit of auxiliary
files, including the XSD and CV documentations as well as ‘How-to’ tutorials
for the example instance documents. All files related to this proposal are
available for download at </span><a href="http://nmrml.org/"><span
class=InternetLink>http://nmrml.org</span></a><span lang=EN-GB>.</span></p>

<p class=MsoNormal>&nbsp;</p>

<p class=Contents1><u><span lang=EN-GB>Contents</span></u></p>

<p class=MsoToc1><span lang=EN-GB>Abstract</span>.......................................................................................................................................... 1</p>

<p class=MsoToc1><span lang=EN-GB>Introduction</span>................................................................................................................................... 3</p>

<p class=MsoToc2>Background............................................................................................................................... 3</p>

<p class=MsoToc2>Relationship to previous formats........................................................................................... 6</p>

<p class=MsoToc2><span lang=EN-GB>Design philosophy</span>................................................................................................................... 6</p>

<p class=MsoToc2>Overall set-up............................................................................................................................ 7</p>

<p class=MsoToc2>Data validation by means of semantic validator software................................................. 8</p>

<p class=MsoToc2><span lang=EN-GB>Tool support and software availability</span>.................................................................................. 9</p>

<p class=MsoToc2>File versioning and release policy...................................................................................... 10</p>

<p class=MsoToc1><span lang=EN-GB>Implementation of the format</span>................................................................................................... 10</p>

<p class=MsoToc2>The nmrML XSD.................................................................................................................... 10</p>

<p class=MsoToc3>The CV term referencing mechanism............................................................................ 12</p>

<p class=MsoToc3>Introduction to nmrML top level elements...................................................................... 13</p>

<p class=MsoToc2>Example nmrML implementations (valid nmrML.xml instances)................................... 16</p>

<p class=MsoToc3>Selecting good example NMR data sets for nmrML xml instances........................... 16</p>

<p class=MsoToc3>Bruker and Varian NMR raw data nmrML examples................................................... 16</p>

<p class=MsoToc2><span lang=EN-GB>Relationship to other efforts and
specifications</span>............................................................... 20</p>

<p class=MsoToc2><span lang=EN-GB>The NMR controlled vocabulary: nmrCV</span>........................................................................... 22</p>

<p class=MsoToc3>Scope and coverage......................................................................................................... 22</p>

<p class=MsoToc3>Development history......................................................................................................... 23</p>

<p class=MsoToc3>Metadata provided with each CV term........................................................................... 23</p>

<p class=MsoToc3>External ontology term reference and import mechanism.......................................... 24</p>

<p class=MsoToc2><span lang=EN-GB>Resolved XSD design Issues</span>.............................................................................................. 24</p>

<p class=MsoToc3><span lang=EN-GB>Count attributes</span>.................................................................................................................. 24</p>

<p class=MsoToc3><span lang=EN-GB>Numerical value and datetimestamp encoding</span>............................................................ 24</p>

<p class=MsoToc2><span lang=EN-GB>The ‘new CV term’ problem</span>.................................................................................................. 25</p>

<p class=MsoToc2><span lang=EN-GB>Other supporting materials</span>................................................................................................... 26</p>

<p class=MsoToc2>Source files and documentation......................................................................................... 26</p>

<p class=MsoToc1>Next steps.................................................................................................................................... 27</p>

<p class=MsoToc2><span lang=EN-GB>How to contribute and provide feedback</span>........................................................................... 28</p>

<p class=MsoToc1>Summary..................................................................................................................................... 28</p>

<p class=MsoToc1>Authors and contributors.......................................................................................................... 28</p>

<p class=MsoToc1>Intellectual property statement................................................................................................. 29</p>

<p class=MsoToc1>Appendix A................................................................................................................................. 29</p>

<p class=Contents1>&nbsp;</p>

<p class=MsoNormal><a href="#_Toc375140733"></a></p>

<h1 style='margin-left:0cm;text-indent:0cm'><a name="_Toc379880873"></a><a
name="_Toc375140734"></a><a name="_Toc156877856"></a><a name="_Toc118017562"></a><a
name="_Ref116882289"></a><span lang=EN-GB>Introduction</span></h1>

<h2><a name="_Toc379880874"></a><a name="_Toc375140735"></a>Background</h2>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph'>Nuclear
magnetic resonance (NMR) spectroscopy is an important analytical method in
metabolomics. As the instrument vendors typically also provide the software to
process the vendor specific data, alternative data analysis software needs to
put considerable efforts into reading and writing these specific vendor
formats. This applies both to commercial software, such as NmrPipe, MestReNova
(Mnova) or Chenomx NMR Suite, but even more so to community developed open
source efforts such as Metaboquant<a href="#_edn1" name="_ednref1" title=""><span
class=EndnoteAnchor><span class=EndnoteAnchor><span style='font-size:12.0pt;
line-height:107%;font-family:"Arial","sans-serif"'>[1]</span></span></span></a>
(Matlab-based), the Batman R package or rNMR. Currently existing standard data
formats such as the JCAMP family<a href="#_edn2" name="_ednref2" title=""><span
class=EndnoteAnchor><span class=EndnoteAnchor><span style='font-size:12.0pt;
line-height:107%;font-family:"Arial","sans-serif"'>[2]</span></span></span></a>
have several drawbacks, especially in metabolomics applications. One problem is
that there is no semantic validation and error checking of JCAMP-DX files.</p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph'>To continue
the standardization efforts by the Metabolomics Standards Initiative (MSI)<span
class=EndnoteAnchor> <a href="#_edn3" name="_ednref3" title=""><span
class=EndnoteAnchor><span style='font-size:12.0pt;line-height:107%;font-family:
"Arial","sans-serif"'>[3]</span></span></a></span> and to leverage and canonize
existing predecessor artifacts<a href="#_edn4" name="_ednref4" title=""><span
class=EndnoteAnchor><span class=EndnoteAnchor><span style='font-size:12.0pt;
line-height:107%;font-family:"Arial","sans-serif"'>[4]</span></span></span></a>
and to coordinate further developments, the COSMOS EU project was granted to
create an open exchange data standard to allow metabolomics NMR raw data, to be
shared and stored in an agreed-upon stable and persistent, yet flexible, vendor
agnostic and easy to understand XML format.</p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph'>The
COSMOS project coordinates efforts from multiple international groups who are
working in NMR based metabolomics and NMR software-engineering to design and
establish an nmrML data format (<a href="http://nmrml.org/"><span
class=InternetLink>http://nmrml.org</span></a>) based on the experience with
the PSI (Proteomics Standards Initiative)<a href="#_edn5" name="_ednref5"
title=""><span class=EndnoteAnchor><span class=EndnoteAnchor><span
style='font-size:12.0pt;line-height:107%;font-family:"Arial","sans-serif"'>[5]</span></span></span></a>
mzML<a href="#_edn6" name="_ednref6" title=""><span class=EndnoteAnchor><span
class=EndnoteAnchor><span style='font-size:12.0pt;line-height:107%;font-family:
"Arial","sans-serif"'>[6]</span></span></span></a> format for mass
spectrometry.</p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph'>A
bird’s eye view on the envisioned nmrML use cases is provided in Fig. 1.</p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=695 height=391 src="NMR-ML1.0_specificationDoc-Dateien/image002.png"></p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph'><b>Figure
1</b>: Illustration of NMR data management facilitation by means of the common
nmrML standard developed in COSMOS</p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph;
line-height:115%'>We have formalized these use cases and requirements which the
new standard should meet in the Unified Modeling Language (UML) use case diagram
(Fig. 2) to illustrate the distinct usages of nmrML in a more standardized
manner.</p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=694 height=551 src="NMR-ML1.0_specificationDoc-Dateien/image003.png"></p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph;
line-height:115%'><b>Figure 2:</b> UML use case diagram illustrating the
multiple application scenarios of the nmrML standard</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal><span lang=EN-GB>The following target objectives can be
defined for the nmrML core format:</span></p>

<p class=MsoNormal style='margin-left:36.0pt;text-indent:-18.0pt'>T1.<span
style='font:7.0pt "Times New Roman"'> </span><i><span lang=EN-GB>The discovery
of relevant results, </span></i><span lang=EN-GB>so that, for example, data
sets in a database or public repository that use a particular technique or
combination of techniques can be identified and studied by experimentalists
during experiment design or data analysis.</span></p>

<p class=MsoNormal style='margin-left:36.0pt;text-indent:-18.0pt'>T2.<span
style='font:7.0pt "Times New Roman"'> </span><i><span lang=EN-GB>The sharing of
best practice</span></i><span lang=EN-GB>, whereby, for example, approaches
that have been successful at analysing low abundance analytes can be captured
alongside the results produced.</span></p>

<p class=MsoNormal style='margin-left:36.0pt;text-indent:-18.0pt'>T3.<span
style='font:7.0pt "Times New Roman"'> </span><i><span lang=EN-GB>The evaluation
of results</span></i><span lang=EN-GB>, whereby, for example, the number and
quality of the spectra recorded from a sample can be assessed in the light of
the experimental conditions.</span></p>

<p class=MsoNormal style='margin-left:36.0pt;text-indent:-18.0pt'>T4.<span
style='font:7.0pt "Times New Roman"'> </span><i>The sharing of data sets in an
open vendor agnostic manner, </i>so that, for example, public repositories can
import or export data, multi-site projects can share results to support
integrated analysis, or meta-analyses can be performed by third parties from
previously published data.</p>

<p class=MsoNormal style='margin-left:36.0pt;text-indent:-18.0pt'>T5.<span
style='font:7.0pt "Times New Roman"'> </span><i>The most comprehensive support
of the instruments output, </i>so that data can be captured [from] all relevant
forms of NMR data representations.</p>

<p class=MsoNormal style='margin-left:36.0pt;text-indent:-18.0pt'>T6.<span
style='font:7.0pt "Times New Roman"'> </span><i>The increase of data
persistence and traceability over time, </i>by delivering a robust storage
format. <span lang=EN-GB>The primary focus of the model is to support long-term
archiving and sharing, rather than day-to-day laboratory management, although
the model is extensible to support context-specific details.</span></p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph'><span
lang=EN-GB>This document defines a specification and is not a tutorial. As
such, the presentation of technical details is deliberately direct. The role of
the text is to describe the schema model and justify design decisions made.
This document does not provide comprehensive examples of the schema in use.
Example documents are provided separately (see nmrml.org/examples) and should
be examined in conjunction with this document. It is anticipated that tutorial
material will be developed to aid implementation. Although the present
specification document describes constraints and guidelines related to the
content of an nmrML document as well as the availability of tools helping to
read and write nmrML, it does not describe any implementation constraints or
specifications such as coding language or operating system for software that
will generate and/or read nmrML data.</span></p>

<h2 style='margin-left:0cm;text-indent:0cm'><a name="_Toc379880875"></a><a
name="_Toc375140736"></a>Relationship to previous formats</h2>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph'>Cosmos
WP2 started the nmrML.xsd development by modification of a predecessor XSD<a
href="#_edn7" name="_ednref7" title=""><span class=EndnoteAnchor><span
class=EndnoteAnchor><span style='font-size:12.0pt;line-height:107%;font-family:
"Arial","sans-serif"'>[7]</span></span></span></a> and with additional elements
and structures from the BML-NMR XSD developed by Christian Ludwig and D.
Rubtsov in Birmingham<a href="#_edn8" name="_ednref8" title=""><span
class=EndnoteAnchor><span class=EndnoteAnchor><span style='font-size:12.0pt;
line-height:107%;font-family:"Arial","sans-serif"'>[8]</span></span></span></a>.
<span lang=EN-GB>Both these efforts were integrated by COSMOS, expanding the
schema of the TMIC Group, as it was already capturing the basic raw data and
had the CV reference mechanism (see Tab 1) already in place. The merged
successor artefact described in this document is now called nmrML and
invalidates both predecessors as the MSI recommended common standard.</span></p>

<h2 style='margin-left:0cm;text-indent:0cm'><a name="_Toc379880876"></a><a
name="_Toc375140737"></a><span lang=EN-GB>Design philosophy</span></h2>

<p class=MsoNormal><span lang=EN-GB>Since the development of nmrML brought
together many different philosophies, the designers agreed on the following
design principles:</span></p>

<ol style='margin-top:0cm' start=1 type=1>
 <li class=MsoNormal><b><span lang=EN-GB>Keep the format simple.</span></b><span
     lang=EN-GB> The data format should be easy for developers to understand
     and integrate into software. This means that it must be handled with off
     the shelve XML and ontology editors. Many elaborate extensions were
     proposed but most were rejected in favour of a simple implementation.</span></li>
 <li class=MsoNormal><b><span lang=EN-GB>Minimize alternate ways of encoding
     the same information</span></b><span lang=EN-GB>. Such flexibility, while
     sometimes touted as a benefit, is bad for data formats.</span></li>
 <li class=MsoNormal><b><span lang=EN-GB>The data format should reference the
     original files</span></b><span lang=EN-GB> for the sake of posterity and
     in the case where original vendor specific information is required.</span></li>
 <li class=MsoNormal><b><span lang=EN-GB>The data format should be flexible</span></b><span
     lang=EN-GB> and allow for multiple use cases of NMR experiments.</span></li>
 <li class=MsoNormal><b><span lang=EN-GB>Build in some flexibility for encoding
     new important information but keep the format stable</span></b><span
     lang=EN-GB>. There is a strong desire from companies that develop software
     for their customers to keep the data format stable over long periods of time.</span></li>
 <li class=MsoNormal><b><span lang=EN-GB>Allow 1D raw data and ultimately processed
     2D NMR spectra to be shared</span></b><span lang=EN-GB> in an open and vendor
     agnostic manner. Record enough information about an NMR spectrum
     acquisition to allow for further processing of the raw spectrum without
     referring to the original vendor files.</span></li>
 <li class=MsoNormal><b><span lang=EN-GB>Allow xml data files to be
     automatically validated on content correctness. </span></b>For example
     minimum information standards like the MIBBI CIMIR<a href="#_ftn1"
     name="_ftnref1" title=""><span class=MsoFootnoteReference><span
     class=MsoFootnoteReference><span style='font-size:12.0pt;line-height:107%;
     font-family:"Arial","sans-serif"'>[1]</span></span></span></a>
     recommendation could be enforced, or a certain Publisher policy.</li>
</ol>

<h2><a name="_Toc379880877"></a><a name="_Toc375140738"></a>Overall set-up</h2>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph'>As
in our PSI mzML role-model, we agreed on implementing a combined exchange
syntax using XML schema and accompanying controlled vocabulary (CV) terms (Fig.
3). The basics of XML and CV based data standardization have been summarized in
a Powerpoint presentation<a href="#_edn9" name="_ednref9" title=""><span
class=MsoEndnoteReference><span class=MsoEndnoteReference><span
style='font-size:12.0pt;line-height:107%;font-family:"Arial","sans-serif"'>[9]</span></span></span></a>
and highlight the general reasons for our nmrML setup. </p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph'>In
areas where the terminology is likely to change faster than the nmrML XSD could
be updated and aligned, branching out from the XSD to CV usage can compensate
for such dynamics in a more flexible way (read more on this issue in Sec 3.1).
The CV can be maintained externally and even in a decentralized manner. For
example, <span lang=EN-GB>new NMR probe types can be</span><span lang=EN-GB> </span><span
lang=EN-GB>represented in an nmrML file by requesting and adding a new
controlled vocabulary term, without the need for a full XSD revision.</span></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal><img border=0 width=696 height=427
src="NMR-ML1.0_specificationDoc-Dateien/image004.png"></p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph'><b>Figure.
3:</b> nmrML consists of an XSD specification, capturing the more data-near and
less variant raw data, and a CV in OWL format capturing the more variant
contextual terminology on NMR as a simple taxonomy. For example when capturing
information about the NMR instrument configuration there are many different
possibilities such as probe heads, auto-samplers, brands, models, etc.</p>

<p class=MsoNormal>&nbsp;</p>

<h2><a name="_Toc379880878"></a><a name="_Toc375140739">Data validation </a>by
means of semantic validator software</h2>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph'>The
XSD-CV set-up allows for multiple data validation levels to be established
(Fig. 4), contributing to data consistency, completeness and overall quality
assurance. </p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph'><span
lang=EN-GB>To avoid the problem of inconsistent usage of vocabulary terms and
the appearance of different dialects encoding the same information, we are
implementing a semantic validator that checks if the CV terms are used
correctly within a particular XML data file. ‘Correctly’ means that the terms
are used in appropriate XML elements and in the correct cardinality. So, the semantic
validator enforces rules not only making sure that the terms are actually found
in a specified CV, but also that the correct terms are used in the correct
location (XML Tag) in the XML document and the required terms are present the
correct number of times, i.e. that there are two filler values allowed for the
SampleTemperature-Element, and that these must come from the Unit Ontologies’
“temperature unit” subtree.</span></p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph'><span
lang=EN-GB>This allows greater flexibility in the schema, but enforces order in
how the CV terms are used. This will require the discipline of using the
semantic validator exploiting validation rules, not just an XML validator for
checking an annotation prior to storage and submission. The result is that new
technologies or information can be accommodated with adjustments to the
controlled vocabulary and validator, not to the schema which hence can stay
stable.</span></p>

<p class=MsoNormal><img border=0 width=693 height=369 id="Grafik 16"
src="NMR-ML1.0_specificationDoc-Dateien/image005.jpg"></p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph'><b>Fig.
4</b>: XML syntax and structural validity of XML instances can be validated by
an XML parser against the XML Schema. Examples include XML element and
attribute positions, order and cardinality. Our proprietary validation tool
exploits mapping files to enforce also semantic validity<a href="#_edn10"
name="_ednref10" title=""><span class=EndnoteAnchor><span class=EndnoteAnchor><span
style='font-size:12.0pt;line-height:107%;font-family:"Arial","sans-serif"'>[10]</span></span></span></a>
of an xml file by specifying which CV terms are allowed in which elements, from
which ontology they should come, as well as their order and cardinality. The
semantic validator to be developed for the next deliverable checks that the
criteria outlined by the mapping file are met in a given XML instance (.nmrML
data file). The mapping file combined with the CV can also be used for
intelligent support in data acquisition, i.e. when creating an interface that
records NMR experiment information, it can populate a drop down menu or an
autocomplete box with plausible CV terms that make sense in this particular
place in the file.<a name=h.jaizcznjwnbj></a><a name=h.xvhoa9johta9></a></p>

<h2><a name="_Toc379880879"></a><a name="_Toc375140740"></a><a
name="_Toc375140741"></a><span lang=EN-GB>Tool support and software
availability</span></h2>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph'><span
lang=EN-GB>One of the aspects that will enable swift adoption is the timely
availability of open source tools that implement the format (e.g. Batman, nNMR,
pynmrML, …). With these tools many users are able to begin using the format
immediately, without coding their own software. To insure that nmrML will be
adopted quickly, the format is presented with several tools that write, read,
and validate nmrML:</span></p>

<ol style='margin-top:0cm' start=1 type=1>
 <li class=MsoNormal><b><span lang=EN-GB>Format converters and parser tools</span></b><span
     lang=EN-GB> that translate the exchange syntax from vendor formats like
     Bruker and Varian to nmrML format.</span></li>
 <li class=MsoNormal><span lang=EN-GB>The <b>nmrML semantic validator</b>,
     which checks for correct implementation of nmrML files, i.e. with respect
     to CV term usage and cardinalities <i>(to be released later)</i>.</span></li>
 <li class=MsoNormal><b><span lang=EN-GB>nmrML bindings</span></b><span
     lang=EN-GB> for multiple programming languages (<a
     name="__DdeLink__2597_211979919">Java, Python, Ruby</a>), as well as for
     widespread data analysis tools like the R statistics package.</span></li>
</ol>

<p class=MsoNormalCxSpLast style='margin-bottom:0cm;margin-bottom:.0001pt;
line-height:115%'>&nbsp;</p>

<h2><a name="_Toc379880880">File versioning and release policy</a></h2>

<p class=MsoNormal>We are finalizing the release policy and hence refer to the
corresponding wiki page draft at <a
href="https://github.com/nmrML/nmrML/wiki/nmrML-release-process">https://github.com/nmrML/nmrML/wiki/nmrML-release-process</a>.
We are working on automatizing the release procedure via a release script.</p>

<p class=MsoNormal>&nbsp;</p>

<h1 style='margin-left:0cm;text-indent:0cm'><a name="_Toc379880881"></a><a
name="_Toc375140742"></a><span lang=EN-GB>Implementation of the format</span></h1>

<h2><a name="_Toc379880882"></a><a name="_Toc375140743"></a><a
name="_Toc156877857"></a><a name="_Toc118017563"></a><a name="_Ref116790912"></a><a
name="_Toc375140744"></a>The nmrML XSD</h2>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph;
line-height:115%'>Any valid nmrML XML instance must adhere to a specific structure
or grammar, which is specified in an XML Schema Definition (XSD). The nmrML XSD
is split up into multiple sections that organize the information that can
appear in nmrML in an intuitive way that facilitates understanding of the
format as well as tool development. The current top level structure of the
nmrML XSD is described in Fig. 5.</p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=490 height=474 src="NMR-ML1.0_specificationDoc-Dateien/image006.png"></p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph;
line-height:115%'><b>Figure 5:</b> The top level XML elements of the nmrML.xsd,
illustrating its main elements and datatypes thereof. The CVList element
defines the controlled vocabularies from which terms are drawn in an XML data
file. The other elements are self-explanatory and refer to the actual NMR data
description, a bit like sections in a material &amp; methods paragraph,
describing an NMR experiment. These are described below. For detailed
documentation, we refer to the HTML documentation of the XSD, or the XSD
itself, in which extensive element annotations explain the usage of the
elements.</p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph;
line-height:115%'>&nbsp;</p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph;
line-height:115%'>At certain locations in the XSD the user is allowed to
describe his data by means of standardized controlled vocabulary terms. The XSD
‘branches out’ into CV-usage, where:</p>

<p class=MsoNormal style='margin-left:36.0pt;text-align:justify;text-justify:
inter-ideograph;text-indent:-18.0pt;line-height:115%'><span style='font-family:
Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>The terms are unstable, variant &amp; dynamically evolving, or
need to be changed and updated often</p>

<p class=MsoNormal style='margin-left:36.0pt;text-align:justify;text-justify:
inter-ideograph;text-indent:-18.0pt;line-height:115%'><span style='font-family:
Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>The terms refer to fast paced dynamically changing terms such as
software names/versions, processing parameters etc.</p>

<p class=MsoNormal style='margin-left:36.0pt;text-align:justify;text-justify:
inter-ideograph;text-indent:-18.0pt;line-height:115%'><span style='font-family:
Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>The terms describe contextual metadata, rather than concrete NMR
raw data, i.e. for cases where the terminology is already extensively defined
in existing ontologies or CVs.</p>

<p class=MsoNormal style='margin-left:36.0pt;text-align:justify;text-justify:
inter-ideograph;text-indent:-18.0pt;line-height:115%'><span style='font-family:
Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>The terms are better maintained by a fast reacting NMR user
community rather than by resource-limited Cosmos work packages.</p>

<p class=MsoNormal style='margin-left:36.0pt;text-align:justify;text-justify:
inter-ideograph;text-indent:-18.0pt;line-height:115%'><span style='font-family:
Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>The terms represent search attributes for data querying and large
scale database-integration in an open linked data fashion.</p>

<p class=MsoNormal style='margin-left:36.0pt;text-align:justify;text-justify:
inter-ideograph;text-indent:-18.0pt;line-height:115%'><span style='font-family:
Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>The terms should be accessible to rule-based or other reasoning techniques
and validation</p>

<p class=MsoNormal style='margin-left:36.0pt;text-align:justify;text-justify:
inter-ideograph;text-indent:-18.0pt;line-height:115%'><span style='font-family:
Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>The terms should be exploited by profiting from robust
subsumption, e.g. exploiting the taxonomic CV backbone to generalize over query
attributes and hence increase result recall and precision.</p>

<h3><a name="_Toc379880883">The CV term referencing mechanism</a></h3>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph;
line-height:115%'>We here outline how CV term usage in nmrML is specified in the
XSD. The requirement and modality for a CV term occurrence in an XML instance
is specified in the XSD by <i>reference elements/types</i> as illustrated in
Table 1. Keep in mind that the last element (UserParamType) captures free text
and makes no CV reference to populate an xml element.</p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph;
line-height:115%'><b>Table 1:</b> The different ways to reference a CV term
from within an nmrML xml file are listed.</p>

<table class=MsoNormalTable border=1 cellspacing=0 cellpadding=0
 style='margin-left:-.25pt;border-collapse:collapse;border:none'>
 <tr style='height:13.2pt'>
  <td width=95 valign=top style='width:70.9pt;border:solid #00000A 1.0pt;
  background:white;padding:0cm 5.4pt 0cm 5.15pt;height:13.2pt'>
  <p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph;
  line-height:115%'><b>Reference Type</b></p>
  </td>
  <td width=246 valign=top style='width:184.25pt;border:solid #00000A 1.0pt;
  border-left:none;background:white;padding:0cm 5.4pt 0cm 5.15pt;height:13.2pt'>
  <p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph;
  line-height:115%'><b>Definition</b></p>
  </td>
  <td width=69 valign=top style='width:52.1pt;border:solid #00000A 1.0pt;
  border-left:none;background:white;padding:0cm 5.4pt 0cm 5.15pt;height:13.2pt'>
  <p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph;
  line-height:115%'><b>Attributes</b></p>
  </td>
  <td width=214 valign=top style='width:160.5pt;border:solid #00000A 1.0pt;
  border-left:none;background:white;padding:0cm 5.4pt 0cm 5.15pt;height:13.2pt'>
  <p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph;
  line-height:115%'><b>Comment</b></p>
  </td>
 </tr>
 <tr style='height:13.2pt'>
  <td width=95 valign=top style='width:70.9pt;border:solid #00000A 1.0pt;
  border-top:none;background:white;padding:0cm 5.4pt 0cm 5.15pt;height:13.2pt'>
  <p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph;
  line-height:115%'><span style='font-size:8.0pt;line-height:115%'>CVTermType</span></p>
  </td>
  <td width=246 valign=top style='width:184.25pt;border-top:none;border-left:
  none;border-bottom:solid #00000A 1.0pt;border-right:solid #00000A 1.0pt;
  background:white;padding:0cm 5.4pt 0cm 5.15pt;height:13.2pt'>
  <p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph;
  line-height:115%'><span style='font-size:8.0pt;line-height:115%'>Elements of
  this type hold additional data or annotation as a simple CV term with no
  further values (Parameters) associated with it. Only controlled CV terms are
  allowed here.</span></p>
  </td>
  <td width=69 valign=top style='width:52.1pt;border-top:none;border-left:none;
  border-bottom:solid #00000A 1.0pt;border-right:solid #00000A 1.0pt;
  background:white;padding:0cm 5.4pt 0cm 5.15pt;height:13.2pt'>
  <p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph;
  line-height:115%'><i><span style='font-size:8.0pt;line-height:115%'>CVRef
  (the ontology), accession (the ID), name (the label)</span></i></p>
  </td>
  <td width=214 valign=top style='width:160.5pt;border-top:none;border-left:
  none;border-bottom:solid #00000A 1.0pt;border-right:solid #00000A 1.0pt;
  background:white;padding:0cm 5.4pt 0cm 5.15pt;height:13.2pt'>
  <p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph;
  line-height:115%'><span style='font-size:8.0pt;line-height:115%'>The “CVRef”
  attribute contains an id for the source CV, unique to the XML instance, that
  is defined in the cvList element. This allows for multiple CVs to be
  referenced crisp and unambiguously. The “accession” attribute contains the ID
  of the CVterm which is unique within the CV. The “name” attribute contains
  the term itself, the intelligible label which is e.g. displayed it to a user)
  without requiring the CV file to be downloaded and parsed.</span></p>
  </td>
 </tr>
 <tr style='height:13.2pt'>
  <td width=95 valign=top style='width:70.9pt;border:solid #00000A 1.0pt;
  border-top:none;background:white;padding:0cm 5.4pt 0cm 5.15pt;height:13.2pt'>
  <p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph;
  line-height:115%'><span style='font-size:8.0pt;line-height:115%'>CVParamType</span></p>
  </td>
  <td width=246 valign=top style='width:184.25pt;border-top:none;border-left:
  none;border-bottom:solid #00000A 1.0pt;border-right:solid #00000A 1.0pt;
  background:white;padding:0cm 5.4pt 0cm 5.15pt;height:13.2pt'>
  <p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph;
  line-height:115%'><span style='font-size:8.0pt;line-height:115%'>Elements of
  this type hold additional data or annotation. In contrast to CVTermType, here
  a pair of descriptors, i.e. the CV term AND a value (=Parameter) is captured.
  Only controlled terms are allowed here.</span></p>
  </td>
  <td width=69 valign=top style='width:52.1pt;border-top:none;border-left:none;
  border-bottom:solid #00000A 1.0pt;border-right:solid #00000A 1.0pt;
  background:white;padding:0cm 5.4pt 0cm 5.15pt;height:13.2pt'>
  <p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph;
  line-height:115%'><span style='font-size:8.0pt;line-height:115%'>CVRef,
  accession, name, <i>value</i></span></p>
  </td>
  <td width=214 valign=top style='width:160.5pt;border-top:none;border-left:
  none;border-bottom:solid #00000A 1.0pt;border-right:solid #00000A 1.0pt;
  background:white;padding:0cm 5.4pt 0cm 5.15pt;height:13.2pt'>
  <p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph;
  line-height:115%'><span style='font-size:8.0pt;line-height:115%'>The ‘value’
  attribute stores the parameter to be captured as value.</span></p>
  </td>
 </tr>
 <tr style='height:13.2pt'>
  <td width=95 valign=top style='width:70.9pt;border:solid #00000A 1.0pt;
  border-top:none;background:white;padding:0cm 5.4pt 0cm 5.15pt;height:13.2pt'>
  <p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph;
  line-height:115%'><span style='font-size:8.0pt;line-height:115%'>CVParamWithUnitType</span></p>
  </td>
  <td width=246 valign=top style='width:184.25pt;border-top:none;border-left:
  none;border-bottom:solid #00000A 1.0pt;border-right:solid #00000A 1.0pt;
  background:white;padding:0cm 5.4pt 0cm 5.15pt;height:13.2pt'>
  <p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph;
  line-height:115%'><span style='font-size:8.0pt;line-height:115%'>Elements of
  this type hold additional data or annotation, i.e. a controlled term
  describing a parameter, as well as a value and a description of the unit the
  value is recorded in. Only controlled values are allowed here. The unit
  ontology is typically used to provide the terms for the unit.</span></p>
  </td>
  <td width=69 valign=top style='width:52.1pt;border-top:none;border-left:none;
  border-bottom:solid #00000A 1.0pt;border-right:solid #00000A 1.0pt;
  background:white;padding:0cm 5.4pt 0cm 5.15pt;height:13.2pt'>
  <p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph;
  line-height:115%'><span style='font-size:8.0pt;line-height:115%'>CVRef,
  accession, name, value, <i>unitCVRef, unitAccession, unitName</i></span></p>
  </td>
  <td width=214 valign=top style='width:160.5pt;border-top:none;border-left:
  none;border-bottom:solid #00000A 1.0pt;border-right:solid #00000A 1.0pt;
  background:white;padding:0cm 5.4pt 0cm 5.15pt;height:13.2pt'>
  <p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph;
  line-height:115%'><span style='font-size:8.0pt;line-height:115%'>The
  ‘unitCvRef’, ‘unitAccession’ and ‘unitName’ attributes are used in the same
  way to describe the unit as the ‘cvRef’, ‘accession’ and ‘name’ terms are
  used to describe other CVTerms.</span></p>
  </td>
 </tr>
 <tr style='height:13.2pt'>
  <td width=95 valign=top style='width:70.9pt;border:solid #00000A 1.0pt;
  border-top:none;background:white;padding:0cm 5.4pt 0cm 5.15pt;height:13.2pt'>
  <p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph;
  line-height:115%'><span style='font-size:8.0pt;line-height:115%'>ValueWithUnitType</span></p>
  </td>
  <td width=246 valign=top style='width:184.25pt;border-top:none;border-left:
  none;border-bottom:solid #00000A 1.0pt;border-right:solid #00000A 1.0pt;
  background:white;padding:0cm 5.4pt 0cm 5.15pt;height:13.2pt'>
  <p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph;
  line-height:115%'><span style='font-size:8.0pt;line-height:115%'>Elements of
  this type hold additional data or annotation. Only controlled values are
  allowed here. For cases where only a Value with an ontologically defined Unit
  should be given. Elements of this type hold a value and a reference to the
  unit the value is recorded in, but is used in locations where the type of
  value is already defined by the element, but the unit of the value still
  needs to be recorded.</span></p>
  </td>
  <td width=69 valign=top style='width:52.1pt;border-top:none;border-left:none;
  border-bottom:solid #00000A 1.0pt;border-right:solid #00000A 1.0pt;
  background:white;padding:0cm 5.4pt 0cm 5.15pt;height:13.2pt'>
  <p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph;
  line-height:115%'><span style='font-size:8.0pt;line-height:115%'>Value,
  unitAccession, unitName, unitCvRef</span></p>
  </td>
  <td width=214 valign=top style='width:160.5pt;border-top:none;border-left:
  none;border-bottom:solid #00000A 1.0pt;border-right:solid #00000A 1.0pt;
  background:white;padding:0cm 5.4pt 0cm 5.15pt;height:13.2pt'>
  <p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph;
  line-height:115%'><span style='font-size:8.0pt;line-height:115%'>&nbsp;</span></p>
  </td>
 </tr>
 <tr style='height:13.2pt'>
  <td width=95 valign=top style='width:70.9pt;border:solid #00000A 1.0pt;
  border-top:none;background:white;padding:0cm 5.4pt 0cm 5.15pt;height:13.2pt'>
  <p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph;
  line-height:115%'><span style='font-size:8.0pt;line-height:115%'>UserParamType</span></p>
  </td>
  <td width=246 valign=top style='width:184.25pt;border-top:none;border-left:
  none;border-bottom:solid #00000A 1.0pt;border-right:solid #00000A 1.0pt;
  background:white;padding:0cm 5.4pt 0cm 5.15pt;height:13.2pt'>
  <p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph;
  line-height:115%'><span style='font-size:8.0pt;line-height:115%'>Elements of
  this type hold uncontrolled user parameters (essentially allowing free text).
  For cases where no suitable CV term exists. Before using these, one should
  verify whether there is an appropriate CV term available, and if so, use the
  CV term instead. This list can however later be exploited to generate corresponding
  term requests in given ontologies or CVs.</span></p>
  </td>
  <td width=69 valign=top style='width:52.1pt;border-top:none;border-left:none;
  border-bottom:solid #00000A 1.0pt;border-right:solid #00000A 1.0pt;
  background:white;padding:0cm 5.4pt 0cm 5.15pt;height:13.2pt'>
  <p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph;
  line-height:115%'><span style='font-size:8.0pt;line-height:115%'>Name, <i>valueType,</i>
  value, unitAccession, unitName, unitCvRef</span></p>
  </td>
  <td width=214 valign=top style='width:160.5pt;border-top:none;border-left:
  none;border-bottom:solid #00000A 1.0pt;border-right:solid #00000A 1.0pt;
  background:white;padding:0cm 5.4pt 0cm 5.15pt;height:13.2pt'>
  <p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph;
  line-height:115%'><span style='font-size:8.0pt;line-height:115%'>The
  ‘valueType’ attribute</span></p>
  </td>
 </tr>
</table>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph;
line-height:115%'><span style='font-size:8.0pt;line-height:115%'>&nbsp;</span></p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph;
line-height:115%'>References to the specific CVs used in the XML instance are
recorded in the ‘cvList’ element at the top of the file, which allows for unambiguous
reference to CV terms.</p>

<h3><a name="_Toc379880884">Introduction to nmrML top level elements</a></h3>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph;
line-height:115%'>The ‘fileDescription’ element captures a general description
about the file and its contents which allows for easy categorization of
different types of nmrML instances e.g. 1D vs. 2D. The ‘contactList’ element
captures information that allows one to contact the original creators/corresponding
authors of a file/paper in the case that further clarification is needed. The
‘sourceFileList’ contains information about the original files used to make the
nmrML instance including files that were required during the acquisition of the
spectrum, for example a Varian processing parameter file or a source code file
for a pulse program. Similarly the ‘softwareList’ element captures references
to software that was used during data acquisition and processing, and may
include several different pieces of software. The
‘informationConfigurationList’ element contains information about the
configuration of an instrument beyond the acquisition parameters, for example
the brand and model of the instrument. The ‘acquisition’ element captures the
processing parameters used during the acquisition. Since vendors have their own
set of names for each of these parameters, we have standardized them with
hopefully intuitive clear names. This element also contains the captured FID
data. The ‘spectrumList’ element contains one or more spectra in the frequency
domain.</p>

<p class=MsoCaption align=center style='text-align:center'><a
name=h.57qe2wpxgoxw></a><a name=h.6lwqdpw6km0x></a><img border=0 width=695
height=405 src="NMR-ML1.0_specificationDoc-Dateien/image007.png"></p>

<p class=MsoCaption style='text-align:justify;text-justify:inter-ideograph;
line-height:115%'><a name=h.uzb4olup7pk8></a><span style='font-size:11.0pt;
line-height:115%'>Figure 6: </span><span style='font-size:11.0pt;line-height:
115%;font-weight:normal'>Specification of CV term usage via the CVParam element
in the XSD (Oxygen Design view screenshot). The accession attribute encodes the
CV term ID and the name encodes the CV term (label).</span></p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph;
line-height:115%'><a name=h.rgp03fins4oz></a>An example of how a CV term
reference by means of the CVParam type (specified in Fig 6) is used in an
example XML instance  data file can be found in Fig. 7. </p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=644 height=425 src="NMR-ML1.0_specificationDoc-Dateien/image008.png"></p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph'><a
name=h.hen641lm2qds></a><b>Figure 7</b>: Example XML instance (Oxygen <i>grid</i>
view) for example data from the original J. Cruz XML example<a href="#_edn11"
name="_ednref11" title=""><span class=EndnoteAnchor><span class=EndnoteAnchor><span
style='font-size:12.0pt;line-height:107%;font-family:"Arial","sans-serif"'>[11]</span></span></span></a>.
This figure illustrates instantiation of CV terms to describe a concrete file
content via CV Parameters. Where the correct entity usage for some values was
not given in the original file or doubtful, value entries were marked with the
String &quot;???&quot;.</p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph'>T<span
style='font-size:11.5pt;line-height:107%;color:black'>he large binary FID data
is stored in the xml at the end of the file in order to allow researchers/programs
to look at the first part of the file, which contains the human readable data,
without the need to load the whole binary blob.</span></p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph'><span
style='font-size:11.5pt;line-height:107%;color:black'>At the core of XML Schema
definitions lies a principle called object inheritance. Like in object oriented
programming languages a complex but basic element can be defined in an abstract
and general way. This allows to derive subelements which inherit the already
defined characteristics of the parent-element, but can have additional
characteristics.</span></p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph'><span
style='font-size:11.5pt;line-height:107%;color:black'>The nmrML.xsd and
data.nmrML XML files can be opened with any text editor and are human readable.
Element descriptions were added as annotations to help rendering the schema
self-explanatory and intuitive. To foster element retrieval and selection, we
added a synonym field per XSD element (as we capture synonyms for the CV).
These annotations are useful user entry points when searching for suitable
annotations/elements.</span></p>

<p class=MsoNormal><span style='font-size:11.5pt;line-height:107%;color:black'>Values
and attributes for elements that appear in an nmrML file can be</span></p>

<p class=MsoListParagraphCxSpFirst style='margin-left:21.0pt;text-indent:-18.0pt'><span
style='font-size:11.5pt;line-height:107%;color:black'>a)<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;
</span></span><span style='font-size:11.5pt;line-height:107%;color:black'>autogenerated
at run-time form the source vendor file parameters via a simple mapping that
the parser<a href="#_ftn2" name="_ftnref2" title=""><span
class=MsoFootnoteReference><span class=MsoFootnoteReference><span
style='font-size:11.5pt;line-height:107%;font-family:"Arial","sans-serif";
color:black'>[2]</span></span></span></a> accesses.</span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:21.0pt;text-indent:-18.0pt'><span
style='font-size:11.5pt;line-height:107%;color:black'>b)<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;
</span></span><span style='font-size:11.5pt;line-height:107%;color:black'>autogenerated
at parser run-time by computation from given parameters in the vendor data
files. An example is the value for gamma_b1_pulse_field_strength, which is
computed<a href="#_ftn3" name="_ftnref3" title=""><span
class=MsoFootnoteReference><span class=MsoFootnoteReference><span
style='font-size:11.5pt;line-height:107%;font-family:"Arial","sans-serif";
color:black'>[3]</span></span></span></a> from the varian pulse with parameter.
</span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:21.0pt;text-indent:-18.0pt'>c)<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span><span
style='font-size:11.5pt;line-height:107%;color:black'>autogenerated totally
anew at parser run time. These refer to mainly administratory information, e.g.
local variables like hashes and temporary IDs.</span></p>

<p class=MsoListParagraphCxSpLast style='margin-left:21.0pt;text-indent:-18.0pt'>d)<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span><span
style='font-size:11.5pt;line-height:107%;color:black'>manually/user generated additional
annotations that pertain to metadata that is not easily accessible or not
available from the vendor files.</span></p>

<p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph'>&nbsp;</p>

<h2><a name="_Toc379880885">Example nmrML implementations</a><a
name=h.ia6spdh4fg5x></a> (valid nmrML.xml instances)</h2>

<p class=MsoNormal>We created multiple example xml files to serve as
data-driven check on the format and allow end-users to learn and grasp the nmrML
format more easily.</p>

<h3><a name="_Toc379880886">Selecting good example NMR data sets for nmrML xml
instances</a></h3>

<p class=MsoNormal>We defined the following characteristics of
intelligible/intuitive example data set in order to increase didactic
compliance:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>The data was gathered in a prototypical, abundant experiment set
up, representative for metabolomics data acquisition</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>The data should stem from a simple experimental set-up (e.g. 1D
1H NMR data)</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>The data has a published paper available (not a method-, but a
research-paper)</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>The data has a database entry available, e.g. in MetaboLights<a
href="#_edn12" name="_ednref12" title=""><span class=EndnoteAnchor><span
class=EndnoteAnchor><span style='font-size:12.0pt;line-height:107%;font-family:
"Arial","sans-serif"'>[12]</span></span></span></a> or HMDB<a href="#_edn13"
name="_ednref13" title=""><span class=EndnoteAnchor><span class=EndnoteAnchor><span
style='font-size:12.0pt;line-height:107%;font-family:"Arial","sans-serif"'>[13]</span></span></span></a></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>The data has accompanying original data files (FIDs)</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>The data is using an abundant vendor format like Bruker, Varian or
JEOL standard files</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>The data is associated with a responsive contact person, in case
someone needs to get back to the data producers to be able to gather additional
information or resolve questions</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>The data has been analyzed further with open source tools like
Batman or MetaboQuant, so that we can later reproduce the same results based on
the converted nmrML data.</p>

<h3><a name="_Toc379880887">Bruker and Varian NMR raw data nmrML examples</a></h3>

<p class=MsoNormal>According to these criteria we have collated example NMR raw
data sets to be converted into nmrML by an automatic parser that reads in the
vendor files and writes valid nmr xml files which comply to the nmrML.XSD. These
examples .nmrML xml files can be found in the corresponding github ‘example’
folder<a href="#_ftn4" name="_ftnref4" title=""><span
class=MsoFootnoteReference><span class=MsoFootnoteReference><span
style='font-size:12.0pt;line-height:107%;font-family:"Arial","sans-serif"'>[4]</span></span></span></a>,
together with an accompanying readme file illustrating its generation in more
detail. For a quick glance you should look under <a
href="http://nmrml.org/examples">http://nmrml.org/examples</a>. To learn nmrML
we advise our readers to look at these example files in the vendor format they
are familiar with. Another place to look at examples are the documentation pages
at nmrml.org/schema.</p>

<p class=MsoNormal><b>Example 1 (Cruz_Example): </b>At first, we analyzed, if
our schema compensated for all data required by the original predecessor. The
original J. Cruz nmrML XML example was taken from <a
href="http://www.metabolomicscentre.ca/nmrML/biosample-concentrations.xml"><span
class=InternetLink>http://www.metabolomicscentre.ca/nmrML/biosample-concentrations.xml</span></a>
and was translated manually into an nmrML XML instance. </p>

<p class=nobreak><b>Example 2 (reference_spectra_example): </b>Another example
was created from a reference spectrum obtained from HMDB (<a
href="http://www.hmdb.ca/spectra/nmr_one_d/1024"><span class=InternetLink>http://www.hmdb.ca/spectra/nmr_one_d/1024</span></a>).
The file was initially written manually, obtaining values from the Varian
procpar file and a python script for encoding the raw FID data into the correct
format. This example also proved useful for creating the conversion software
since the output could be compared.</p>

<p class=nobreak><b>Example 3 (IPB_HOP_Example): </b>We added another example
which represents a typical metabolomics experimental set-up. Thirteen hop plant
ecotypes were profiled for interesting secondary metabolites using MS and NMR
in combination<a href="#_edn14" name="_ednref14" title=""><span
class=EndnoteAnchor><span class=EndnoteAnchor><span style='font-size:12.0pt;
line-height:107%;font-family:"Arial","sans-serif"'>[14]</span></span></span></a>.
Fig. 7 illustrates how 1D acquisition and raw FID data is stored in an nmrML
xml instance for one of the hop variants (AHTM). In order to familiarize with
nmrML, this example is the one you should start with when you are working with
Varian data.</p>

<p class=nobreak>After the development of the vendor to nmrML parsers was
sufficiently complete, more example files were generated, i.e. from
MetaboLights<a href="#_ftn5" name="_ftnref5" title=""><span
class=MsoFootnoteReference><span class=MsoFootnoteReference><span
style='font-size:12.0pt;line-height:107%;font-family:"Arial","sans-serif"'>[5]</span></span></span></a>
entries MTBLS1 and 25 data. All examples can be browsed in the corresponding
GitHub folder.</p>

<p class=nobreak><b>Example 4 (MTBLS1): </b>This is example nmrML file that was
generated from Bruker data.<b> </b>If you are a Bruker user then this is your
entry point to familiarize with nmrML.</p>

<p class=nobreak>&nbsp;</p>

<p class=nobreak><img border=0 width=691 height=522 id="Grafik 1"
src="NMR-ML1.0_specificationDoc-Dateien/image009.png"></p>

<p class=MsoNormal><img border=0 width=690 height=522 id=Picture
src="NMR-ML1.0_specificationDoc-Dateien/image010.png"></p>

<p class=MsoNormal><b>Figure 7:</b> We here provide an nmrML XSD snippet where
the FID element is shown (above). The code screenshot (below) illustrates how
basic 1D acquisition parameters are stored in the example XML and how Varian
raw FID data is stored. The FID is stored as a binary blob (base64 encoded
binary data). Byte ordering is always Intel-style little endian. Computers
using a different endian style must convert to/from little endian when
writing/reading nmrML. The FID should be converted into an array of complex
numbers before encoding.</p>

<h2 style='margin-left:0cm;text-indent:0cm'><a name="_Toc379880888"></a><a
name="_Toc375140745"></a><a name="_Ref116790953"></a><a name="_Toc118017564"></a><a
name="_Toc156877858"><span lang=EN-GB>Relationship to other efforts and specifications</span></a></h2>

<p class=MsoNormal><span lang=EN-GB>The specification described in this
document is not developed in isolation; indeed, it is designed to be
complementary to, and thus used in conjunction with several existing and
emerging information models. Related specifications include the following:</span></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='margin-left:39.0pt;text-indent:-18.0pt'>1.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span><b>nmrML parsers</b>:
We have implemented multiple parsers. The python based vendor2nmrML format
parser<a href="#_ftn6" name="_ftnref6" title=""><span
class=MsoFootnoteReference><span class=MsoFootnoteReference><span
style='font-size:12.0pt;line-height:107%;font-family:"Arial","sans-serif"'>[6]</span></span></span></a>
uses the nmrGlue API to access the most important parameters from the Bruker
and Varian Files. Our Parser, developed by Michael Wilson writes these
parameters into an nmrML xml file. Another parser, called nmrRIO, parses nmrML
and makes its content available to R based statistics tools such as Batman and
rNMR.</p>

<p class=MsoNormal style='margin-left:39.0pt;text-indent:-18.0pt'>2.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span><b>nmrQuant:</b>
A future extension of the nmrML schema that allows to capture NMR based
quantification data on metabolites</p>

<p class=MsoNormal style='margin-left:39.0pt;text-indent:-18.0pt'>3.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span><b>nmrIdent:</b>
An future extension of the nmrML schema that allows to capture NMR based
Identification data on metabolites.</p>

<p class=MsoNormal style='margin-left:39.0pt;text-indent:-18.0pt'>4.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span><b>nmrML (to
nmrCV) mapping file:</b> Although the structure of the CV and the nmrML schema
are related, the details of which CV terms are allowed/recommended at a given
schema location is reported in an external mapping file. The mapping file is a
list of associations between a cvParam element in the schema (described via XPath)
and the allowed CV term entry nodes/branches of the CV terms expected in that
location. The mapping file is read and interpreted by the semantic validator in
order to check that the data annotation is consistent and complete. The mapping
file needs to be checked and eventually updated when the CV terms or structure
are changed. A mapping file will be provided in the next version of this
document</p>

<p class=MsoNormal style='margin-left:39.0pt;text-indent:-18.0pt'>5.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span><b><span
lang=EN-GB>BioSharing</span></b><span lang=EN-GB>:</span><span lang=EN-GB> </span><a
href="http://www.biosharing.org"><span lang=EN-GB>http://www.biosharing.org</span></a><span
lang=EN-GB>  is a consortium that collects and promotes common representations
for Minimum Information Standards for Metabolomics. We will build on the BioSharing
and the ISA-Tab efforts to harmonize representation of the metadata
recommendations with other -omics communities, and use automated tests to
ensure the interoperability of the metadata between the involved data
producers, -consumers and -repositories.</span></p>

<p class=MsoNormal style='margin-left:39.0pt;text-indent:-18.0pt'>6.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span><b>CIMR (Core
Information for Metabolomics Reporting)</b> at <a
href="http://mibbi.sourceforge.net/projects/CIMR.shtml">http://mibbi.sourceforge.net/projects/CIMR.shtml</a><span
class=InternetLink><i> </i></span>specifies the minimal guidelines reporting
metabolomics<a href="#_edn15" name="_ednref15" title=""><span
class=MsoEndnoteReference><span class=MsoEndnoteReference><span
style='font-size:12.0pt;line-height:107%;font-family:"Arial","sans-serif"'>[15]</span></span></span></a>
work in a comprehensive and repeatable manner:<i> </i></p>

<p class=MsoNormal style='margin-left:39.0pt;text-indent:-18.0pt'>7.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span><b>Minimum
Information on an NMR experiment (</b><b><span lang=EN-GB>MI NMR)</span></b> as
defined in <a
href="http://link.springer.com/article/10.1007%2Fs11306-006-0040-4"><span
class=InternetLink>http://link.springer.com/article/10.1007%2Fs11306-006-0040-4</span></a><span
lang=EN-GB>. The COSMOS nmrML format has been designed to encode the
requirements specified in this MI NMR. However, nmrML does not enforce MI NMR
compliance itself; nmrML documents may be valid and useful without being fully
MI NMR compliant. The nmrML validator will have settings to validate either the
basic nmrML raw data level or at a more comprehensive MI NMR compliant level,
depending on the needs of the user.</span></p>

<p class=MsoNormal style='margin-left:39.0pt;text-indent:-18.0pt'>8.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span><b><span
lang=EN-GB>SMRS (Standard Metabolic Reporting Structures)</span></b><span
lang=EN-GB>: An Open Standard for Reporting Metabolic Data (http://www.smrsgroup.org).</span></p>

<p class=MsoNormal style='margin-left:39.0pt;text-indent:-18.0pt'>9.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span><b>Analytical
Information Markup Language (AnIML)</b>:An emerging ASTM XML standard for
analytical chemistry data. <a href="http://animl.sourceforge.net">http://animl.sourceforge.net</a>
AniML captures the instrumental set-up of chemical analytical techniques, so is
more general in scope than our NMR restricted effort.</p>

<p class=MsoNormal style='margin-left:39.0pt;text-indent:-18.0pt'>10.<span
style='font:7.0pt "Times New Roman"'> </span><b>mzML</b>: <a
href="http://www.psidev.info/mzml_1_0_0">http://www.psidev.info/mzml_1_0_0</a>.<b>
</b>The XML based open and PSI approved Mass Spectrometry data standard. This
format served as a general role model for the development of nmrML and its
success made us to copycat many of their design decisions.</p>

<p class=MsoNormal style='margin-left:39.0pt;text-indent:-18.0pt'>11.<span
style='font:7.0pt "Times New Roman"'> </span><b>ISA-Tab</b>: <a
href="http://isa-tools.org/">http://isa-tools.org</a>. As research in
biomedical and life sciences is increasingly moving towards multi-omics
studies, the ‘Investigation/Study/Assay’ ISA-Tab format was developed to
represent experimental metadata independently from the assay technology used.
We will use ISA-Tab to standardize metabolomics reporting requirements and
terminologies through customized ISA configurations.</p>

<p class=MsoNormal>&nbsp;</p>

<h2 style='margin-left:0cm;text-indent:0cm'><a name="_Toc379880889"></a><a
name="_Toc375140746"><span lang=EN-GB>The NMR controlled vocabulary</span></a><span
lang=EN-GB>: nmrCV</span></h2>

<h3><a name="_Toc379880890">Scope and coverage</a></h3>

<p class=MsoNormal>This controlled vocabulary (CV) is an MSI-approved ontology
developed under COSMOS governance. This CV was derived from two predecessors
(The NMR CV from the David Wishard Group, developed by Joseph Cruz) and the MSI
NMR CV developed by Daniel Schober at the EBI. This simple taxonomy of terms<a
href="#_ftn7" name="_ftnref7" title=""><span class=MsoFootnoteReference><span
class=MsoFootnoteReference><span style='font-size:12.0pt;line-height:107%;
font-family:"Arial","sans-serif"'>[7]</span></span></span></a> serves the
nuclear magnetic resonance markup language (nmrML) with meaningful descriptors
to amend the nmrML XML in an explicit yet standardized way. Metabolomics
scientists are encouraged to use this CV to annotate their NMR raw and
experimental context data.</p>

<p class=MsoNormal>The nmrCV is implemented as a simple taxonomy in owl syntax<a
href="#_ftn8" name="_ftnref8" title=""><span class=MsoFootnoteReference><span
class=MsoFootnoteReference><span style='font-size:12.0pt;line-height:107%;
font-family:"Arial","sans-serif"'>[8]</span></span></span></a>. We choose the
OWL Syntax<a href="#_edn16" name="_ednref16" title=""><span
class=EndnoteAnchor><span class=EndnoteAnchor><span style='font-size:12.0pt;
line-height:107%;font-family:"Arial","sans-serif"'>[16]</span></span></span></a>
over the OBO format<a href="#_edn17" name="_ednref17" title=""><span
class=EndnoteAnchor><span class=EndnoteAnchor><span style='font-size:12.0pt;
line-height:107%;font-family:"Arial","sans-serif"'>[17]</span></span></span></a>
as exchange syntax for the CV, as the OBO tools are less stable. Because the
OBO format is only established in the biology domain (lack of off-the-shelf
development tools) and there are hence less resources to integrate with.</p>

<p class=MsoNormal>The nmrCV is accessible at <a href="http://nmrml.org/cv">http://nmrml.org/cv</a>,
but soon will be available from ontology libraries such as the NCBO Bioportal<a
href="#_edn18" name="_ednref18" title=""><span class=MsoEndnoteReference><span
class=MsoEndnoteReference><span style='font-size:12.0pt;line-height:107%;
font-family:"Arial","sans-serif"'>[18]</span></span></span></a>. More extensive
documentation will be available form the website soon.</p>

<h3><a name="_Toc379880891">Development history</a></h3>

<p class=MsoNormal>After agreement on the set up of development tools (Protégé
4), we formulated our CV design principles, namely agreed on file names, format
syntax, namespaces, (auto) term ID schemes, a term obsoletion policy, as well
as versioning &amp; release procedures. We analyzed existing CVs on suitability
and modelling errors<a href="#_edn19" name="_ednref19" title=""><span
class=EndnoteAnchor><span class=EndnoteAnchor><span style='font-size:12.0pt;
line-height:107%;font-family:"Arial","sans-serif"'>[19]</span></span></span></a>.
From the given predecessor CVs, we proceeded in a bottom-up and middle-out
approach to expand the CV. We first added CV terms as required in the XSD
leafs, i.e. where CVTermType, CVParamType, CVParamWithUnitType references occur
in XSD elements. After this, we continued with a use-case driven term
population, taking multiple data sets as examples. No high throughput
term-additions were attempted in the early design phase, as this would clutter
the CV with terms of doubtful need, impair orientation in the term tree as too
many terms distract us from getting the main structure right. The nmrCV.owl
ontology momentarily contains ~ 600 classes under the nmr: namespace. Around
2000 terms are imported from the Unit Ontology and the BFO top level ontology.</p>

<h3><a name="_Toc379880892">Metadata provided with each CV term</a></h3>

<p class=MsoNormal>Not all of our terms currently have natural language
definitions, as these are time-intensive and not needed for our use case, given
the terms are usually self-explanatory. None has deeper provenance data
explicitly annotated (there is only an implicit indication from which
predecessor CV a term came in the ID ranges). We try to avoid getting stuck in
the meta-ether, and have been pragmatic about this.</p>

<p class=MsoNormal>The CV in its current stage is far from being complete. Its
coverage is just sufficient to express the example raw data. We rely on our
users that code their own NMR data in nmrML to submit request-emails for
missing nmr CV terms to the nmrML mailing list<a href="#_ftn9" name="_ftnref9"
title=""><span class=MsoFootnoteReference><span class=MsoFootnoteReference><span
style='font-size:12.0pt;line-height:107%;font-family:"Arial","sans-serif"'>[9]</span></span></span></a>.</p>

<p class=MsoNormal><span lang=EN-GB>The nmrCV should be dynamically maintained
via the COSMOS nmrML mailing list (</span><a
href="https://groups.google.com/forum/?hl=en#!forum/nmrml/join"><span
class=InternetLink><span lang=EN-GB>https://groups.google.com/forum/?hl=en#!forum/nmrml/join</span></span></a><span
lang=EN-GB> ) to allow any user to request new terms in agreement with the community
involved. Once a consensus is reached among the community the new terms are
added within few days. If there is no obvious consensus, the CV coordinators
should vote and make a decision.</span></p>

<p class=MsoNormal>If you require us to integrate a whole list of new nmr CV terms,
you can do so in an email to the author. In this case a term batch-submission
table should have the following fields:</p>

<p class=MsoNormal><span style='font-family:"Courier New"'>term name
(rdfs:label)</span><span style='font-family:Wingdings'>à</span><span
style='font-family:"Courier New"'>skos:prefLabel,ideally adhering to labelling
best practice<a href="#_edn20" name="_ednref20" title=""><span
class=MsoEndnoteReference><span class=MsoEndnoteReference><span
style='font-size:12.0pt;line-height:107%;font-family:"Courier New"'>[20]</span></span></span></a>.</span></p>

<p class=MsoNormal><span style='font-family:"Courier New"'>term definition in natural
language (IAO_0000115)</span><span style='font-family:Wingdings'>à</span><span
style='font-family:"Courier New"'>skos:definition</span></p>

<p class=MsoNormal><span style='font-family:"Courier New"'>superclass (ideally
a term from the current nmrCV.owl, or an own suggestion)</span></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>Optional fields (good to have) are:</p>

<p class=MsoNormal><span style='font-family:"Courier New"'>synonym
(oboInOwl:hasExactSynonym)</span><span style='font-family:Wingdings'>à</span><span
style='font-family:"Courier New"'>skos:altLabel</span></p>

<p class=MsoNormal><span style='font-family:"Courier New"'>term definition
source (dc:source)</span></p>

<p class=MsoNormal><span style='font-family:"Courier New"'>dc:creator</span><span
style='font-family:Wingdings'>à</span><span style='font-family:"Courier New"'>dc:author</span></p>

<p class=MsoNormal><a name=h.jrgepkhd66wu></a><span style='font-family:"Courier New"'>example
of usage</span><span style='font-family:Wingdings'>à</span><span
style='font-family:"Courier New"'>skos:example</span></p>

<p class=MsoNormal><a name=h.36nn271n2c7p></a>&nbsp;</p>

<h3><a name="_Toc379880893">External ontology term reference and import
mechanism</a></h3>

<p class=MsoNormal>We use owl:import statements to import the BFO top level and
the unit ontology into the nmrCV. Although this option tends to clutter the core
CV with seldomly used terms and occupies a lot of RAM, it retains all metadata
for the imported terms. We might at some point use the MIREOT term import
mechanism, i.e. to reference Chebi terms for the description of Atoms,
chemicals and small molecules.</p>

<h2 style='margin-left:0cm;text-indent:0cm'><a name="_Toc379880894"></a><a
name="_Toc375140747"></a><span lang=EN-GB>Resolved XSD design Issues</span></h2>

<p class=MsoNormal><span lang=EN-GB>There were several issues regarding the
design of the format that were not clear cut, and a design choice was made that
was not completely agreeable to everyone. So that these issues do not keep
coming up, we document here the issues and why the decision that is implemented
was made.</span></p>

<h3 style='margin-left:0cm;text-indent:0cm'><a name="_Toc379880895"></a><a
name="_Toc375140748"></a><span lang=EN-GB>Count attributes</span></h3>

<p class=MsoNormal><span lang=EN-GB>At the moment all list elements would have
a count attribute. The reason is that parsers implemented in languages where
memory allocation or array sizing is important, it is a nice performance
enhancement to have a count attribute indicating how many elements there are in
the list. As this is an easy target for creating inconsistent files (i.e.
specifying a count=”5” attribute followed by 6 items in the list), this was
deemed to be not very useful. That is where we diverge from our role model the
PSI mzML not using count attributes in the next versions.</span></p>

<h3 style='margin-left:0cm;text-indent:0cm'><a name="_Toc379880896"></a><a
name="_Toc375140749"></a><span lang=EN-GB>Numerical value and datetimestamp
encoding</span></h3>

<p class=MsoNormal><span lang=EN-GB>All numerical values shall appear in the
XML schema datatype specification (http://www.w3.org/TR/xmlschema-2/). The
number 1/10 must always appear as 0.10 and never as 0,10. A preceeding + before
a number (+5.0) is prohibited.</span></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal><span lang=EN-GB>Datetimestamps must also be encoded as in
the XML specification such as 2007-06-27T15:23:45.00035.</span></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal><span lang=EN-GB>All id attributes follow the XML schema
datatype xs:ID (http://www.w3.org/TR/xmlschema-2/#ID), which means that no two
id attributes may be the same within a document, and id attributes must be
purely alphanumerical strings with at least one letter. Thus, they may not
contain spaces or underscores, and id attributes may not be a plain number.</span></p>

<h2 style='margin-left:0cm;text-indent:0cm'><a name="_Toc379880897"></a><a
name="_Toc375140750"></a><span lang=EN-GB>The ‘new CV term’ problem</span></h2>

<p class=MsoNormal><span lang=EN-GB>Whereas we allow the users to specify CV
terms and CV term with unit and value combinations along the term reference
modalities described in Table 1, all information encoded as element attributes
are never controlled vocabulary terms. Thus, as an example to describe spectrum
type, the cvParam element must be provided to specify a term below NMR:1002007
“NMR spectrum”, such as:</span></p>

<p class=MsoNormal><span lang=EN-GB style='font-size:8.0pt;line-height:107%;
font-family:"Courier New"'>&lt;cvParam cvLabel=&quot;nmrCV&quot; accession=&quot;</span><span
lang=EN-GB> </span><span lang=EN-GB style='font-size:8.0pt;line-height:107%;
font-family:"Courier New"'>NMR:1000118&quot; name=&quot;J-resolved spectrum&quot;
value=&quot;&quot;/&gt;</span></p>

<p class=MsoNormal><span lang=EN-GB>Note that both accession and the term name
are provided. Parsers should focus on the accession number as this should never
change, even if the term name is adjusted in the controlled vocabulary later.
Note that there is no value. The mere presence of the term is the annotation.</span></p>

<p class=MsoNormal><span lang=EN-GB>The problem comes when there is a new term
to be added. Let us assume that it becomes necessary to add a new spectrum type
“heteronuclear J-resolved spectrum”. Vendor X would like to start writing nmrML
with this spectrum type. What should happen and what could also happen?</span></p>

<p class=MsoNormal><span lang=EN-GB>In an ideal world, Vendor X would contact
the Cosmos standards development Work package 2 nmrML list and request a new
child term of “J-resolved spectrum”. A CV coordinator would verify that this is
a new concept, not simply a synonym of an existing concept, add the term to the
CV and release a new version of the nmrCV.owl file at the same location. Vendor
X would obtain the accession number and could begin writing out valid nmrML. The
semantic validator would (and already does) automatically download the new .owl
file and validates that the new nmrML is semantically valid using the new term.</span><span
lang=EN-GB> </span><span lang=EN-GB>If the file is then distributed to
arbitrary site Y, local software will suddenly encounter this new term:</span></p>

<p class=MsoNormal><span lang=EN-GB style='font-size:8.0pt;line-height:107%;
font-family:"Courier New"'>&lt;cvParam cvLabel=&quot;MS&quot; accession=&quot;</span><span
lang=EN-GB> </span><span lang=EN-GB style='font-size:8.0pt;line-height:107%;
font-family:"Courier New"'>NMR:1000129 &quot; name=&quot;</span><span
lang=EN-GB> </span><span lang=EN-GB style='font-size:8.0pt;line-height:107%;
font-family:"Courier New"'>heteronuclear J-resolved spectrum&quot;
value=&quot;&quot;/&gt;</span></p>

<p class=MsoNormal><span lang=EN-GB>and there is the possibility of failure.
The first problem that may occur is that reader software may try to understand
what the spectrum type is, but it will not find a spectrum type that it
understands. Therefore it can only conclude that either no spectrum type was
provided, or one of the terms it doesn’t recognize is a spectrum type but it
won’t know which one. If the software could connect to the Internet, and could
automatically download the latest .obo file, and look to see if this term was
in the file, and then determine what parent the above term had, and understand
that the parent is “NMR spectrum”, then the software could conclude that the
above cv term is an NMR spectrum, but a new one that it doesn’t not know how to
handle yet. Such a string of logic is not terribly difficult but it is not
trivial and is objectionable to some.</span></p>

<p class=MsoNormal><span lang=EN-GB>Worse yet, Vendor X could have been lazy
and not even contacted the nmrML CV coordinators and just started publishing nmrML
with (“option A”):</span></p>

<p class=MsoNormal><span lang=EN-GB style='font-size:8.0pt;line-height:107%;
font-family:"Courier New"'>&lt;cvParam cvLabel=&quot;nmrCV&quot;
accession=&quot;NMR:9999999&quot; name=&quot; heteronuclear J-resolved spectrum&quot;
value=&quot;&quot;/&gt;</span></p>

<p class=MsoNormal><span lang=EN-GB>There is no way to resolve this. No reader
could possibly know how to handle this. It should be avoided at all cost.
Although it should be noted that until the proper accession number is
furnished, such an approach could at least initially be used. In any case,
processing software may still not know how to properly handle heteronuclear
J-resolved spectra, but it should already be able to confidently understand
what the spectrum type is and admit it cannot handle it.</span></p>

<h2><a name="_Toc379880898"></a><a name="_Toc375140751"></a><span lang=EN-GB>Other
supporting materials</span></h2>

<p class=MsoNormal><span lang=EN-GB>This document cannot be fully judged on its
own. It is important to study the accompanying power point presentations, sample
instance documents, controlled vocabulary, schema files, and the software that
implements this pre-release version of nmrML. </span></p>

<h2><a name="_Toc379880899">Source files and documentation</a></h2>

<p class=MsoNormal>All these files and programs are available from nmrml.org.
Further development versions with accompanying readme files can be found on the
nmrML GitHub site (<a href="https://github.com/nmrML/nmrML">https://github.com/nmrML/nmrML</a>).</p>

<p class=MsoNormal>The following describes the more important files and
documents that we have prepared and their respective download locations:</p>

<p class=MsoNormal><b>nmrML.xsd</b></p>

<p class=MsoNormal><b>(nmrml.org/schema/1.0.rc1/nmrML.xsd):</b> An XML schema
that defines the structure, content and parts of the semantics of the allowed
nmrML XML documents. The XML schema definition (XSD) uses XML Schema version
1.1 format following the W3C recommendation (w3.org/XML/Schema). The schema
allows for the capture of raw NMR spectrum data and acquisition parameters for
both one-dimensional and two-dimensional spectra, including two-dimensional
J-resolved spectra.</p>

<p class=MsoNormal><b>nmrCV.owl</b></p>

<p class=MsoNormal><b>(nmrml.org/cv/1.0.rc1/nmrCV.owl): </b>The controlled
vocabulary (CV) describing the more variant terminology in an unambiguous and
standardized way. This ontology is the MSI-sanctioned successor of artifacts
developed previously at EMBL-EBI, Hinxton, UK (D. Schober, Sansone Group) and
the Wishart Research Group, Edmonton, Canada (J. Cruz). This CV currently
covers the description of NMR spectrum acquisition set up and raw data
generated during the acquisition. There is less coverage of data generated by
analysis of the spectrum such as metabolite quantification and identification.
The CV terms are used within the nmrML xml file, at positions specified in the
XSD, e.g. by CVParam references.</p>

<p class=MsoNormal><b>XML example files</b></p>

<p class=MsoNormal><b>(https://github.com/nmrML/nmrML/tree/master/examples/working.tmp/nmrML
&amp; https://github.com/nmrML/nmrML/tree/master/examples): </b>Multiple XML
instances complying with the XSD were generated to illustrate the usage of
nmrML in a practical experiment data annotation. These instances also served to
test the XSD and CV on coverage, structural soundness and to test parser
software.</p>

<p class=MsoNormal><b>XSDToCV mapping file</b></p>

<p class=MsoNormal><b>(nmrml.org/schema/1.0.rc1/nmrml-mapping.xml): </b>This
xml file specifies rules to constrain CV term usage during data entry, i.e. it
allows to verify validness of CV term usage in the nmrML XML files. This
mapping file will also be used to enforce minimal metadata standards<a
href="#_edn21" name="_ednref21" title=""><span class=EndnoteAnchor><span
class=EndnoteAnchor><span style='font-size:12.0pt;line-height:107%;font-family:
"Arial","sans-serif"'>[21]</span></span></span></a>. Only a very first draft
has been created for testing purposes.</p>

<p class=MsoNormal><b>HTML documentation files</b></p>

<p class=MsoNormal><b>(nmrml.org/schema/1.0.rc1/doc &amp;
nmrml.org/cv/1.0.rc1/doc): </b>Documentation was generated with automated tools
that describes the nmrML XSD and the CV OWL. The documentation allows non-XML
and non-ontology savvy end-users to open, browse and comment on the standards
as well as facilitating the use of the data format by developers and the
implementation of tools that use, read or write nmrML.</p>

<p class=MsoNormal>All source files are available on the project Github pages,
together with an accompanying readme file.</p>

<p class=MsoNormal><b>GitHub:                                  </b><a
href="https://github.com/nmrML/nmrML"><span class=InternetLink>https://github.com/nmrML/nmrML</span></a></p>

<p class=MsoNormal><b>Cosmos website:               </b><a
href="http://www.cosmos-fp7.eu/"><span class=InternetLink>http://www.cosmos-fp7.eu</span></a></p>

<p class=MsoNormal><b>nmrML website:                  </b><a
href="http://nmrml.org/"><span class=InternetLink>http://nmrml.org</span></a></p>

<p class=MsoNormal><b>nmrML wiki:                         </b><a
href="http://cosmos-fp7.eu/nmrML/"><span class=InternetLink>http://cosmos-fp7.eu/nmrML/</span></a></p>

<p class=MsoNormal><b>nmrML google forum:       </b><a
href="https://groups.google.com/forum/"><span class=InternetLink>https://groups.google.com/forum/#!forum/nmrml</span></a></p>

<h1 style='margin-left:0cm;text-indent:0cm'><a name="_Toc379880900">Next steps</a></h1>

<p class=MsoNormal>Further testing of the XSD is required with diverse
experimental configurations, to ensure that our goal of flexibility has been
achieved. Continuing to improve the documentation and building a community of
users will hopefully provide further feedback for improvements to the Schema
and CV. At the same time we will continue the data-driven CV expansions and add
new terms according to the additional examples selected by our different
partners. We must also ensure that the schema is compatible with the steps we
are taking toward NMR QuantML and IdentML, which pose the next layer in our
onion approach to expand the nmrML (from raw data to preprocessed data to post
processed data to full spectra, and their ident and Quant annotations).</p>

<p class=MsoNormal>Also we need to work out an evaluation pipeline. As part of
the next deliverable (D2.5 - Real data, Converters, Validators and Parsers for
NMR-ML, m24), we will implement the CV-aware validator software and extensive
mapping files containing the verification rules to check XML instances on
semantic errors and completeness.</p>

<p class=MsoNormal>In parallel we will implement the parsers for format
conversions and I/O to open source tools. The creation of ISA Tab
specifications for easy tabular data entry and minimal reporting requirement
enforcement is considered a further next step (D2.6).</p>

<h2 style='margin-left:0cm;text-indent:0cm'><a name="_Toc379880901"></a><a
name="_Toc375140752"></a><a name="_Toc375140753"></a><a name="_Toc375140754"></a><a
name="_Toc375140756"></a><span lang=EN-GB>How to contribute and provide
feedback</span></h2>

<p class=MsoNormal><span lang=EN-GB>We here explicitly solicit feedback on our
Project, i.e. on potential XSD and CV errors like missing elements, unintuitive
element labels or unintuitive element nesting. We also solicit feedback on the
didactics of our specification documentation. Ideally you could start to test
the overall format by making your Bruker or Varian NMR raw data accessible to
our format converter and check the resulting nmrML format. Experienced xml users
can also write valid nmrML files from scratch, validating it through the XSD,
and annotate the nmrML further (manually).</span></p>

<p class=MsoNormal><span lang=EN-GB>Questions of interest to the whole
community and more general requests should be send to the COSMOS nmrML mailing
list at </span><a
href="https://groups.google.com/forum/?hl=en#!forum/nmrml/join"><span
class=InternetLink><span lang=EN-GB>https://groups.google.com/forum/?hl=en#!forum/nmrml/join</span></span></a><span
lang=EN-GB> </span></p>

<p class=MsoNormal><span lang=EN-GB>Remarks, change- and expansion requests on
the nmrML.xsd, nmrCV and the documentation should be send to the email list at </span><a
href="mailto:info@nmrml.org"><span lang=EN-GB>info@nmrml.org</span></a></p>

<p class=MsoNormal><span lang=EN-GB>If you have a specific request and are a
Git user you can use the nmrML Git issue tracker at </span></p>

<p class=MsoNormal><a
href="https://github.com/nmrML/nmrML/issues?direction=desc&amp;sort=created&amp;state=open"><span
lang=EN-GB>https://github.com/nmrML/nmrML/issues?direction=desc&amp;sort=created&amp;state=open</span></a><span
lang=EN-GB> </span></p>

<p class=MsoNormal><span lang=EN-GB>Here use a „CV:“-prefix in the issues
subject-line to ease issue archaeology. For XSD requests it should be „XSD:“
(the default), „EX:“ for data examples, “PC:” for Parsers &amp; Conversters and
“VAL:” for validation Issues.</span></p>

<h1 style='margin-left:0cm;text-indent:0cm'><a name="_Toc379880902"></a><a
name="_Toc375140759"></a>Summary</h1>

<p class=MsoNormal><span lang=EN-GB>This document contains the specification
for using the nmrML format to represent NMR spectrometry raw data and
associated contextual metadata. This specification, in conjunction with the nmrML
XML Schema and nmrCV, constitutes the proposed Metabolomics Standards
Initiative sanctioned NMR exchange data coordinated by the COSMOS EU project. For
didactic compliance, we provide nmrML example data files and illustrate how
these were generated from the vendor files and how they can be validated.</span></p>

<p class=MsoNormal>&nbsp;</p>

<h1 style='margin-left:0cm;text-indent:0cm'><a name="_Toc379880903"></a><a
name="_Toc375140760"></a>Authors and contributors</h1>

<p class=MsoNormal><a name="_Toc118017570"></a><a name="_Toc111817895"></a><span
lang=PT-BR>&nbsp;</span></p>

<p class=MsoNormal><span lang=PT-BR>Daniel Schober, Mike Wilson, Steffen
Neumann for Cosmos WP 2</span></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal><span lang=EN-GB>We acknowledge the following people, who contributed
to the model or controlled vocabulary development and gave feedback or tested
nmrML:</span></p>

<p class=MsoNormal>Reza Salek, Mike Wilson, Steffen Neumann, Christian Ludwig,
Ulrich Günter, Catherine Deborde, Daniel Joseph, Antonio Rosato, John Easton,
Tim Ebbels, Kenneth Haug, Annick Moing, Luis de Figueiredo, Philippe Rocca
Serra, Jie Hao, Andrea Porzel</p>

<p class=MsoNormal>&nbsp;</p>

<h1 style='margin-left:0cm;text-indent:0cm'><a name="_Toc379880904"></a><a
name="_Toc375140761"></a><a name="_Toc375140762"></a><a name="_Toc156877875"></a><a
name="_Toc155584023"></a><a name="_Toc153690678"></a><a name="_Toc526008660"></a>Intellectual
property statement</h1>

<p class=MsoNormal>COSMOS and MSI take no position regarding the validity or
scope of any intellectual property or other rights that might be claimed to
pertain to the implementation or use of the technology described in this
document or the extent to which any license under such rights might or might
not be available; neither does it represent that it has made any effort to
identify any such rights. Copies of claims of rights made available for
publication and any assurances of licenses to be made available, or the result
of an attempt made to obtain a general license or permission for the use of
such proprietary rights by implementers or users of this specification can be
obtained from the MSI/PSI Chair.</p>

<p class=MsoNormal>COSMOS and MSI invite any interested party to bring to its
attention any copyrights, patents or patent applications, or other proprietary
rights, which may cover technology that may be required to practice this
recommendation. Please address the information to the main author.</p>

<p class=MsoNormal>&nbsp;</p>

<h1 style='margin-left:0cm;text-indent:0cm'><a name="_Toc379880905"></a><a
name="_Toc375140763"></a>Appendix A</h1>

<h1 style='margin-left:0cm;text-indent:0cm'>&nbsp;</h1>

<p class=MsoNormal><a name="_Toc375140764"></a><a name="_Toc156877876"></a><a
name="_Toc155584024"></a><a name="_Toc153687291"></a><u>Copyright notice:</u></p>

<p class=MsoNormal>Copyright (C) Cosmos and the Metabolomics Standards
Initiative (2013). All Rights Reserved.</p>

<p class=MsoNormal>This document and translations of it may be copied and
furnished to others, and derivative works that comment on or otherwise explain
it or assist in its implementation, may be prepared, copied, published and
distributed, in whole or in part, without restriction of any kind, provided
that the above copyright notice and this paragraph are included on all such
copies and derivative works. </p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal><a name=31></a><a name=30></a><a name=29></a><u>Disclaimer:</u></p>

<p class=MsoNormal>This document and the information contained herein is
provided on an &quot;AS IS&quot; basis and THE METABOLOMICS STANDARDS
INITIATIVE AND COSMOS DEVELOPERS DISCLAIM ALL WARRANTIES, EXPRESSED OR IMPLIED,
INCLUDING BUT NOT LIMITED TO ANY WARRANTY THAT THE USE OF THE INFORMATION
HEREIN WILL NOT INFRINGE ANY RIGHTS OR ANY IMPLIED WARRANTIES OF
MERCHANTABILITY OR FITNESS FOR A PARTICULAR PURPOSE.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal><b>References</b></p>

</div>

<div><br clear=all>

<hr align=left size=1 width="33%">

<div id=ftn1>

<p class=MsoFootnoteText><a href="#_ftnref1" name="_ftn1" title=""><span
class=MsoFootnoteReference><span class=MsoFootnoteReference><span
style='font-size:10.0pt;line-height:107%;font-family:"Arial","sans-serif"'>[1]</span></span></span></a>
<a href="http://mibbi.sourceforge.net/projects/CIMR.shtml">http://mibbi.sourceforge.net/projects/CIMR.shtml</a>
</p>

</div>

<div id=ftn2>

<p class=MsoFootnoteText><a href="#_ftnref2" name="_ftn2" title=""><span
class=MsoFootnoteReference><span class=MsoFootnoteReference><span
style='font-size:10.0pt;line-height:107%;font-family:"Arial","sans-serif"'>[2]</span></span></span></a>
<a
href="https://github.com/nmrML/nmrML/tree/master/tools/Parser_and_Converters/python/pynmrml"><span
style='font-size:11.5pt;line-height:107%'>https://github.com/nmrML/nmrML/tree/master/tools/Parser_and_Converters/python/pynmrml</span></a><span
style='font-size:11.5pt;line-height:107%;color:black'> </span></p>

</div>

<div id=ftn3>

<p class=MsoNormal><a href="#_ftnref3" name="_ftn3" title=""><span
class=MsoFootnoteReference><span class=MsoFootnoteReference><span
style='font-size:12.0pt;line-height:107%;font-family:"Arial","sans-serif"'>[3]</span></span></span></a>
<span style='font-size:11.5pt;line-height:107%;color:black'>In the varian
reader at nmrML\tools\Parser_and_Converters\python\pynmrml\io\readers we find
the formula “def gamma_b1_pulse_field_strength(self): return 1/( 4 * float(self.pulse_width())
/ 1000000.0 )</span></p>

</div>

<div id=ftn4>

<p class=MsoFootnoteText><a href="#_ftnref4" name="_ftn4" title=""><span
class=MsoFootnoteReference><span class=MsoFootnoteReference><span
style='font-size:10.0pt;line-height:107%;font-family:"Arial","sans-serif"'>[4]</span></span></span></a>
<a href="https://github.com/nmrML/nmrML/tree/master/examples">https://github.com/nmrML/nmrML/tree/master/examples</a>
, look at the files in the latest nmrMLs folders. </p>

</div>

<div id=ftn5>

<p class=MsoFootnoteText><a href="#_ftnref5" name="_ftn5" title=""><span
class=MsoFootnoteReference><span class=MsoFootnoteReference><span
style='font-size:10.0pt;line-height:107%;font-family:"Arial","sans-serif"'>[5]</span></span></span></a>
<a href="http://www.ebi.ac.uk/metabolights">http://www.ebi.ac.uk/metabolights</a>
</p>

</div>

<div id=ftn6>

<p class=MsoFootnoteText><a href="#_ftnref6" name="_ftn6" title=""><span
class=MsoFootnoteReference><span class=MsoFootnoteReference><span
style='font-size:10.0pt;line-height:107%;font-family:"Arial","sans-serif"'>[6]</span></span></span></a>
<a
href="https://github.com/nmrML/nmrML/tree/master/tools/Parser_and_Converters/python/pynmrml"><span
style='font-size:11.5pt;line-height:107%'>https://github.com/nmrML/nmrML/tree/master/tools/Parser_and_Converters/python/pynmrml</span></a></p>

</div>

<div id=ftn7>

<p class=MsoFootnoteText><a href="#_ftnref7" name="_ftn7" title=""><span
class=MsoFootnoteReference><span class=MsoFootnoteReference><span
style='font-size:10.0pt;line-height:107%;font-family:"Arial","sans-serif"'>[7]</span></span></span></a>
No DL semantics used, for a short overview on the differences of CV, taxonomy
and ontology look at <a
href="http://infogrid.org/trac/wiki/Reference/PidcockArticle">http://infogrid.org/trac/wiki/Reference/PidcockArticle</a>
</p>

</div>

<div id=ftn8>

<p class=MsoFootnoteText><a href="#_ftnref8" name="_ftn8" title=""><span
class=MsoFootnoteReference><span class=MsoFootnoteReference><span
style='font-size:10.0pt;line-height:107%;font-family:"Arial","sans-serif"'>[8]</span></span></span></a>
without leveraging on complicated description logics semantics</p>

</div>

<div id=ftn9>

<p class=MsoFootnoteText><a href="#_ftnref9" name="_ftn9" title=""><span
class=MsoFootnoteReference><span class=MsoFootnoteReference><span
style='font-size:10.0pt;line-height:107%;font-family:"Arial","sans-serif"'>[9]</span></span></span></a>
<a href="https://groups.google.com/forum/?hl=en#!forum/nmrml">https://groups.google.com/forum/?hl=en#!forum/nmrml</a>
</p>

</div>

</div>

<div><br clear=all>

<hr align=left size=1 width="33%">

<div id=edn1>

<p class=MsoEndnoteText><a href="#_ednref1" name="_edn1" title=""><span
class=MsoEndnoteReference><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:107%;font-family:"Arial","sans-serif"'>[1]</span></span></span></a><span
class=MsoEndnoteReference>                                              </span>
Wolfram Gronwald, Matthias Klein and Peter Oefner (submitted ?), MetaboQuant: A
Tool Combining Individual Peak Calibration and Outlier Detection for Accurate
Quantification from NMR Spectra</p>

</div>

<div id=edn2>

<p class=MsoEndnoteText><a href="#_ednref2" name="_edn2" title=""><span
class=MsoEndnoteReference><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:107%;font-family:"Arial","sans-serif"'>[2]</span></span></span></a><span
class=MsoEndnoteReference>                                              </span>
P Lampen, J Lambert, RJ Lancashire et al., AN EXTENSION TO THE JCAMP-DX
STANDARD FILE FORMAT, JCAMP-DX V.5.01, Pure Appl. Chem., Vol. 71, No. 8, pp.
1549-1556, 1999</p>

</div>

<div id=edn3>

<p class=MsoEndnoteText><a href="#_ednref3" name="_edn3" title=""><span
class=MsoEndnoteReference><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:107%;font-family:"Arial","sans-serif"'>[3]</span></span></span></a><span
class=MsoEndnoteReference>                                              </span>
Sansone SA,  Schober D, Atherton HJ, Fiehn O, Jenkins H, Rocca-Serra P, Rubtsov
DV, Spasic I, Soldatova L, Taylor C, Tseng A, Viant MR (2007)  Metabolomics
standards initiative: ontology working group work in progress.  Metabolomics 3,
249-256. ISSN 1573-3882</p>

</div>

<div id=edn4>

<p class=MsoEndnoteText><a href="#_ednref4" name="_edn4" title=""><span
class=MsoEndnoteReference><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:107%;font-family:"Arial","sans-serif"'>[4]</span></span></span></a><span
class=MsoEndnoteReference>                                              </span>
Sansone,S.A., Fan,T., Goodacre,R. et al. (2007) The metabolomics standards
initiative. Nat. Biotechnol., 25, 846–848.</p>

</div>

<div id=edn5>

<p class=MsoEndnoteText><a href="#_ednref5" name="_edn5" title=""><span
class=MsoEndnoteReference><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:107%;font-family:"Arial","sans-serif"'>[5]</span></span></span></a><span
class=MsoEndnoteReference>                                              </span>
<a href="http://www.psidev.info/"><span class=InternetLink>http://www.psidev.info/</span></a></p>

</div>

<div id=edn6>

<p class=MsoEndnoteText><a href="#_ednref6" name="_edn6" title=""><span
class=MsoEndnoteReference><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:107%;font-family:"Arial","sans-serif"'>[6]</span></span></span></a><span
class=MsoEndnoteReference>                                              </span>
Martens,L., Chambers,M., Sturm,M. et al. (2011) mzML—a community standard for
mass spectrometry data. Mol. Cell Proteomics, 10, R110000133.
http://www.ncbi.nlm.nih.gov/pubmed/20716697</p>

</div>

<div id=edn7>

<p class=MsoEndnoteText><a href="#_ednref7" name="_edn7" title=""><span
class=MsoEndnoteReference><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:107%;font-family:"Arial","sans-serif"'>[7]</span></span></span></a><span
class=MsoEndnoteReference>                                              </span>
<a href="http://www.metabolomicscentre.ca/exchangeformats.htm"><span
class=InternetLink>http://www.metabolomicscentre.ca/exchangeformats.htm</span></a></p>

</div>

<div id=edn8>

<p class=MsoEndnoteText><a href="#_ednref8" name="_edn8" title=""><span
class=MsoEndnoteReference><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:107%;font-family:"Arial","sans-serif"'>[8]</span></span></span></a><span
class=MsoEndnoteReference>                                              </span>
Taylor CF, Field D, Sansone SA, et al., Promoting coherent minimum reporting
guidelines for biological and biomedical investigations: the MIBBI project, Nat
Biotechnol. 2008 Aug;26(8):889-96. doi: 10.1038/nbt.1411. , PMID:18688244</p>

</div>

<div id=edn9>

<p class=MsoEndnoteText><a href="#_ednref9" name="_edn9" title=""><span
class=MsoEndnoteReference><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:107%;font-family:"Arial","sans-serif"'>[9]</span></span></span></a>
<a
href="https://github.com/nmrML/nmrML/blob/master/docs/SchemaDocumentation/COSMOS_nmrML_Basics.pptx">https://github.com/nmrML/nmrML/blob/master/docs/SchemaDocumentation/COSMOS_nmrML_Basics.pptx</a>
</p>

</div>

<div id=edn10>

<p class=MsoEndnoteText><a href="#_ednref10" name="_edn10" title=""><span
class=MsoEndnoteReference><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:107%;font-family:"Arial","sans-serif"'>[10]</span></span></span></a><span
class=MsoEndnoteReference>                                            </span>
Montecchi-Palazzi L., Kerrien S., Reisinger F. et al. (2009) The PSI semantic
validator: a framework to check MIAPE compliance of proteomics data.
Proteomics, 9, 5112–5119.</p>

</div>

<div id=edn11>

<p class=MsoEndnoteText><a href="#_ednref11" name="_edn11" title=""><span
class=MsoEndnoteReference><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:107%;font-family:"Arial","sans-serif"'>[11]</span></span></span></a><span
class=MsoEndnoteReference>                                            </span> <a
href="http://www.metabolomicscentre.ca/nmrML/biosample-concentrations.xml"><span
class=InternetLink><span style='font-size:11.0pt;line-height:107%'>http://www.metabolomicscentre.ca/nmrML/biosample-concentrations.xml</span></span></a></p>

</div>

<div id=edn12>

<p class=MsoEndnoteText><a href="#_ednref12" name="_edn12" title=""><span
class=MsoEndnoteReference><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:107%;font-family:"Arial","sans-serif"'>[12]</span></span></span></a><span
class=MsoEndnoteReference>                                            </span>
Haug, K., Salek, R. M., Conesa, P., Hastings, J., de Matos, P., Rijnbeek, M.,
et al. (2013). MetaboLights--an open-access general-purpose repository for
metabolomics studies and associated meta-data. [Research Support, Non-U.S.
Gov't]. <i>Nucleic acids research, 41</i>(Database issue), D781-786,
doi:10.1093/nar/gks1004.</p>

</div>

<div id=edn13>

<p class=MsoEndnoteText><a href="#_ednref13" name="_edn13" title=""><span
class=MsoEndnoteReference><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:107%;font-family:"Arial","sans-serif"'>[13]</span></span></span></a><span
class=MsoEndnoteReference>                                            </span>
Wishart, D. S., Jewison, T., Guo, A. C., Wilson, M., Knox, C., Liu, Y., et al.
(2013). HMDB 3.0--The Human Metabolome Database in 2013. [Research Support,
Non-U.S. Gov't]. <i>Nucleic acids research, 41</i>(Database issue), D801-807,
doi:10.1093/nar/gks1065.</p>

<p class=MsoEndnoteText><span class=MsoEndnoteReference>                        </span> Rubtsov
DV, Jenkins H, Ludwig C, Easton J, Viant MR, Günther U, Griffin JL, Hardy N
(2007) Proposed reporting requirements for the description of NMR-based
metabolomics experiments. Metabolomics 3, 223–229.</p>

</div>

<div id=edn14>

<p class=MsoEndnoteText><a href="#_ednref14" name="_edn14" title=""><span
class=MsoEndnoteReference><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:107%;font-family:"Arial","sans-serif"'>[14]</span></span></span></a><span
class=MsoEndnoteReference>                                            </span>
Farag, M., Porzel, A., Schmidt, J. &amp; Wessjohann, L. Metabolite profiling
and fingerprinting of commercial cultivars of Humulus lupulus L. (hop) - a
comparision of MS and NMR methods in metabolomics Metabolomics 8, 492-507,
(2012)</p>

</div>

<div id=edn15>

<p class=MsoEndnoteText><a href="#_ednref15" name="_edn15" title=""><span
class=MsoEndnoteReference><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:107%;font-family:"Arial","sans-serif"'>[15]</span></span></span></a>
Sumner LW, Amberg A, Barrett D, Beale MH, Beger R, et al. 2007. Proposed
minimum reporting standards for chemical analysis. Metabolomics 3:211–21</p>

</div>

<div id=edn16>

<p class=MsoEndnoteText><a href="#_ednref16" name="_edn16" title=""><span
class=MsoEndnoteReference><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:107%;font-family:"Arial","sans-serif"'>[16]</span></span></span></a><span
class=MsoEndnoteReference>                                            </span> <a
href="http://www.w3.org/TR/owl2-syntax/"><span class=InternetLink>http://www.w3.org/TR/owl2-syntax/</span></a></p>

</div>

<div id=edn17>

<p class=MsoEndnoteText><a href="#_ednref17" name="_edn17" title=""><span
class=MsoEndnoteReference><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:107%;font-family:"Arial","sans-serif"'>[17]</span></span></span></a><span
class=MsoEndnoteReference>                                            </span> <a
href="http://www.geneontology.org/GO.format.obo-1_2.shtml"><span
class=InternetLink>http://www.geneontology.org/GO.format.obo-1_2.shtml</span></a></p>

</div>

<div id=edn18>

<p class=MsoEndnoteText><a href="#_ednref18" name="_edn18" title=""><span
class=MsoEndnoteReference><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:107%;font-family:"Arial","sans-serif"'>[18]</span></span></span></a>
<a href="http://bioportal.bioontology.org/">http://bioportal.bioontology.org/</a>
</p>

</div>

<div id=edn19>

<p class=MsoEndnoteText><a href="#_ednref19" name="_edn19" title=""><span
class=MsoEndnoteReference><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:107%;font-family:"Arial","sans-serif"'>[19]</span></span></span></a><span
class=MsoEndnoteReference>                                            </span>
Schober D., Mayer G., Moing A., Eisenacher M., Neumann S., Ontological analysis
of controlled vocabularies used in PSI/MSI supported XML standards, Workshop:
ODLS 2013, GI-Edition Lecture Notes in Informatics, Proceedings of the
Jahrestagung der Gesellschaft für Informatik 2013, Matthias Horbach (Hrsg.),
Koblenz, Germany, 16.–20. September 2013, p. 1875-1888,
https://wiki.imise.uni-leipzig.de/Gruppen/OBML/Workshops/2013-ODLS-en</p>

</div>

<div id=edn20>

<p class=MsoEndnoteText><a href="#_ednref20" name="_edn20" title=""><span
class=MsoEndnoteReference><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:107%;font-family:"Arial","sans-serif"'>[20]</span></span></span></a>
<a href="http://www.obofoundry.org/wiki/index.php/FP_012_naming_conventions"><span
class=InternetLink>http://www.obofoundry.org/wiki/index.php/FP_012_naming_conventions</span></a></p>

</div>

<div id=edn21>

<p class=MsoEndnoteText><a href="#_ednref21" name="_edn21" title=""><span
class=MsoEndnoteReference><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:107%;font-family:"Arial","sans-serif"'>[21]</span></span></span></a><span
class=MsoEndnoteReference>                    </span><span
class=MsoEndnoteReference><span lang=DE>                        </span></span><span
lang=DE> Montecchi-Palazzi L., Kerrien S., Reisinger F. et al. </span>(2009)
The PSI semantic validator: a framework to check MIAPE compliance of proteomics
data. Proteomics, 9, 5112–5119.</p>

</div>

</div>

</body>

</html>
